<!DOCTYPE html>
<html>

<head>

<meta charset="utf-8">

<title>test</title>


<script type="text/javascript" language="javascript" src='https://code.jquery.com/jquery-3.5.1.js'></script>

<script type="text/javascript" language="javascript" src='https://cdn.datatables.net/1.10.22/js/jquery.dataTables.min.js'></script>

<script type="text/javascript" language="javascript" src='https://cdn.datatables.net/fixedcolumns/3.3.2/js/dataTables.fixedColumns.min.js'></script>

<script type="text/javascript" > 


var openTab;

//document ready
$(document).ready(function() {
				var height = $(window).height();
				var h = Math.floor(height - 250);
				var filterHash = new Object();
				var keyType = "3";
				var keyAnnotID = "0";
				var fullMode = 'full';	
				var dblClickMode = 'off';	
				var oldStart = 0;

				//input FILTER
				var head = '#tabFULLSPLIT thead';
				$(head+' tr').clone(true).appendTo(head);
	
				$(head+' tr:eq(1) th').each( function (i) {
					$(this).removeClass('sorting').removeClass('sorting_asc').removeClass('sorting_desc');
					$(this).unbind();
					let title = $(this).text();
					$(this).html( '<input id="col'+i+'" type="text" placeholder="Search" data-index="'+i+'" />' );
				}); //END FILTER



				//INITIALISATION
				var tabFULLSPLIT = $('#tabFULLSPLIT').DataTable({
					stateSave: true,
					stateDuration: 0,
					order: [],
					paging: true,
					orderCellsTop: true,
					scrollX: true,
					scrollY: h,
					//scrollCollapse: true,
					oLanguage: { sLengthMenu: 'Show _MENU_ lines',sInfo: 'Showing _START_ to _END_ of _TOTAL_ lines' },
					lengthMenu: [
							[50, 100, -1],
							[50, 100, 'All']
						],
					fixedColumns:   {
						leftColumns: 1,
						heightMatch: 'auto',
					 },
					stateLoadParams: function( settings, data ) {
						filterHash = data.filter;
						fullMode = data.fullMode ;	
						dblClickMode = data.dblClickMode ;	
						//console.log('load__');
					},
					stateSaveParams: function( settings, data ) {
						data.filter = filterHash;	
						data.fullMode = fullMode;	
						data.dblClickMode = dblClickMode;	
						//console.log('save__');
					},
					drawCallback: function (o) {
						var newStart = this.api().page.info().start;
						if ( newStart != oldStart ) {
							$('.dataTables_scrollBody').scrollTop(0);
							oldStart = newStart;
						}
					},
			});  //END initialisation

			//FILTER FUNCTION
			$(tabFULLSPLIT.table().container() ).on( 'keyup change', 'thead input', function (){ 

						//console.log('toto1');
						var expr = this.value;
						var i = $(this).data('index');
						if(/^[!<>=]+$/.test(expr)){
							return;
						}
						
						if(expr === ''){
							delete filterHash[i] ;
						}else{

							var exprClean = expr.replace(/\s*/g, '');
							filterHash[i] = new Object();
							filterHash[i]['raw'] = exprClean;
							if (/^[!<>=]/.test(expr)) {
								var oper = expr.match(/^([!<>=]+)/);
								var exp = exprClean.match(/^\W+([-]?\w+[.]?\w*)/i);
								filterHash[i]['operator'] = oper[0];
								if (exp === null){
									return;
								}
								filterHash[i]['expr'] = exp[1];

							}else{
								filterHash[i]['operator'] = '==';
								filterHash[i]['expr'] = new RegExp(`${exprClean}`,'i');
							}
						}

						tabFULLSPLIT.draw();
					
					});
					

			//button tab		
			var tabs = `<div class="tab" style="float: left;">
								<button id="FULLbutton" class="tablinks active" onclick="openTab(event, 'full')">COMPACT</button>
								<button id="FULLSPLITbutton" class="tablinks" onclick="openTab(event, 'fullsplit')">EXPANDED</button>
							</div>`;
			$('#tabFULLSPLIT_wrapper').prepend(tabs);
			

				//double click expand
				$('#tabFULLSPLIT tbody').on('dblclick', 'tr', function () {
					var rowID = this.id;			
					var fm = fullMode;
					if (rowID !== '' && fullMode === 'full'){

						if (dblClickMode == 'off'){
							dblClickMode = 'on';
							$('#tabFULLSPLIT_wrapper .DTFC_LeftWrapper thead tr:eq(1) th:eq('+keyAnnotID+') input', $('.tooltipHeader td')[keyAnnotID] ).val(rowID);
							$('#tabFULLSPLIT_wrapper .dataTables_scrollHead thead tr:eq(1) th:eq('+keyType+') input', $('.tooltipHeader td')[keyType] ).val('' );
							filterHash[keyAnnotID] = new Object();
							filterHash[keyAnnotID]['raw'] = rowID;
							filterHash[keyAnnotID]['operator'] = '==';
							filterHash[keyAnnotID]['expr'] = new RegExp(`${filterHash[keyAnnotID]['raw']}`,'i');
							delete filterHash[keyType] ;

						}else{
							dblClickMode = 'off';
							$('#tabFULLSPLIT_wrapper .dataTables_scrollHead thead tr:eq(1) th:eq('+keyType+') input', $('.tooltipHeader td')[keyType] ).val('full' );
							$('#tabFULLSPLIT_wrapper .DTFC_LeftWrapper thead tr:eq(1) th:eq('+keyAnnotID+') input', $('.tooltipHeader td')[keyAnnotID] ).val('');
							filterHash[keyType] = new Object();
							filterHash[keyType]['raw'] = 'full';
							filterHash[keyType]['operator'] = '==';
							filterHash[keyType]['expr'] = new RegExp(`${filterHash[keyType]['raw']}`,'i');
							delete filterHash[keyAnnotID] ;

						}
						
						tabFULLSPLIT.draw();

					}
				} ); //END DOUBLE CLICK

				//FILTERING FUNCTION PUSH AT INITIALISATION
				$.fn.dataTableExt.afnFiltering.push(
					function( oSettings, aData, iDataIndex ) {
						var filtHash = filterHash;
						var filtBool = true;
						for (var keys in filtHash){
							if (filtHash.hasOwnProperty(keys)) {
								var row_data = aData[keys];
			
								switch (filtHash[keys]['operator']){
									case '>': if(row_data > filtHash[keys]['expr']) {filtBool = true;continue;}else{ return false;}
									case '<': if(row_data < filtHash[keys]['expr']) {filtBool = true;continue;}else{ return false;}
									case '>=': if(row_data >= filtHash[keys]['expr']) {filtBool = true;continue;}else{ return false;}
									case '<=': if(row_data <= filtHash[keys]['expr']) {filtBool = true;continue;}else{ return false;}
									case '!=': if(row_data != filtHash[keys]['expr']) {filtBool = true;continue;}else{ return false;}
									case '<>': if(row_data != filtHash[keys]['expr']) {filtBool = true;continue;}else{ return false;}
									case '=': if(row_data == filtHash[keys]['expr']) {filtBool = true;continue;}else{ return false;}
									case '==': var m = row_data.match(filtHash[keys]['expr']); if(m === null){return false;}else{filtBool = true;continue;}
								}
							}
						}
			
						return filtBool;
					}); //END function filterByExp




				//Restore state				
				var state = tabFULLSPLIT.state.loaded();						 
				if ( state  ) {
					//console.log('state__');
					for (var keys in filterHash){
						if (filterHash.hasOwnProperty(keys)) {
							$('#tabFULLSPLIT_wrapper .dataTables_scrollHead thead tr:eq(1) th:eq('+keys+') input', $('.tooltipHeader td')[keys] ).val( filterHash[keys]['raw'] );
							if (filterHash[keys]['operator'] == '=='){
								filterHash[keys]['expr'] = new RegExp(`${filterHash[keys]['raw']}`,'i');
							}
						}
					}
					if (fullMode == 'fullsplit'){
						$('#FULLbutton').removeClass('active');
						$('#FULLSPLITbutton').addClass('active');
                                        }
					if (dblClickMode == 'on'){
						$('#tabFULLSPLIT_wrapper .DTFC_LeftWrapper thead tr:eq(1) th:eq('+keyAnnotID+') input', $('.tooltipHeader td')[keyAnnotID] ).val(filterHash[keyAnnotID]['raw']);
					}

					
					
					tabFULLSPLIT.draw();
				}else{  //END restore
					
					//FULL view on load	
					window.onload =	$('#tabFULLSPLIT_wrapper .dataTables_scrollHead thead tr:eq(1) th:eq('+keyType+') input', $('.tooltipHeader td')[keyType] ).val('full' ).change();			
				}


		


	//click full split button
	openTab =	function(evt, cityName) {
				
				if(cityName === 'full'){
				
					fullMode = 'full';

					filterHash[keyType] = new Object();
					filterHash[keyType]['raw'] = cityName;
					filterHash[keyType]['operator'] = '==';
					filterHash[keyType]['expr'] = new RegExp(`${filterHash[keyType]['raw']}`,'i');
					
					$('#FULLSPLITbutton').removeClass('active');
					$('#FULLbutton').addClass('active');
					
					$('#tabFULLSPLIT_wrapper .dataTables_scrollHead thead tr:eq(1) th:eq('+keyType+') input', $('.tooltipHeader td')[keyType] ).val(cityName );
					
				}else{
					
					fullMode = 'fullsplit';

					delete filterHash[keyType] ;
					$('#FULLbutton').removeClass('active');
					$('#FULLSPLITbutton').addClass('active');
					$('#tabFULLSPLIT_wrapper .dataTables_scrollHead thead tr:eq(1) th:eq('+keyType+') input', $('.tooltipHeader td')[keyType] ).val('');

				}
				
				if (dblClickMode == 'on'){
					dblClickMode = 'off';
					$('#tabFULLSPLIT_wrapper .DTFC_LeftWrapper thead tr:eq(1) th:eq('+keyAnnotID+') input', $('.tooltipHeader td')[keyAnnotID] ).val('');
					delete filterHash[keyAnnotID] ;
				}
	
				
				$('#tabFULLSPLIT').DataTable().draw();
			}


			



});  //END document ready




</script>

<link rel="stylesheet" type="text/css" href='https://cdn.datatables.net/1.10.22/css/jquery.dataTables.min.css'>

<link rel="stylesheet" type="text/css" href='https://cdn.datatables.net/fixedcolumns/3.3.2/css/fixedColumns.dataTables.min.css'>


<style>
/* Style the tab */
	.tab {
		overflow: hidden;
		border: 1px solid #ccc;
		background-color: #f1f1f1;
		left: 0;
		bottom: 0;
		height: 27px;
		position: fixed;
		position: -webkit-sticky;
		position: sticky;
		margin-right: 10px;
		}

/* Style the buttons inside the tab */
	.tab button {
		background-color: inherit;
		float: left;
		border: none;
		outline: none;
		cursor: pointer;
		transition: 0.3s;
		font-size: 90%;
		height: 27px;
		}
/* Change background color of buttons on hover */
	.tab button:hover {
		background-color: #ddd;
		}

/* Create an active/current tablink class */
	.tab button.active {
		background-color: #ccc;
		}
	.tab button:focus {
		background-color: #ccc;
		}

	td {
		text-align: center;
		}
	thead input {
		width: 100%;
		}

	#tabFULLSPLIT.display td.sorting_1{
		background-color: inherit;
	}

	table.dataTable.display tbody tr.odd > .sorting_1, table.dataTable.order-column.stripe tbody tr.odd > .sorting_1{
		background-color: inherit; 
	}
	table.dataTable.display tbody tr.even > .sorting_1, table.dataTable.order-column.stripe tbody tr.even > .sorting_1{
		background-color: inherit; 
	}



/* Style the tab content */
	.tabcontent {
		display: block;
		padding: 6px 12px;
		/*border: 1px solid #ccc;*/
		border-top: none;
		}

	.tooltipHeader{
		position: relative;
		font-size: 90%;
		cursor: help;
	}		
	.tooltipHeader .tooltiptext{
		visibility: hidden;
		width: auto;
		min-width: 250px;
		max-width: 800px;
		height: auto;
		background-color: #1e1e1e;
		color: #fff;
		text-align: left;
		border-radius: 6px;
		padding: 5px 5px;
		position: absolute;
		z-index: 1;
		top: -5px;
		opacity: 0;
		transition: opacity 0.3s;
		text-overflow: ellipsis;
		white-space: normal;
		overflow-wrap: break-word;
		}
	.tooltipHeader:hover .tooltiptext{
		visibility: visible;
		opacity: 1;
	}	

	.tooltip {
		position: relative;
		display: inline-block;
		/*border-bottom: 1px dotted black;*/
		min-width: 100px;
		max-width: 300px;
		overflow-wrap: break-word;
		font-size: 70%;
		}
	
	.tooltip .tooltiptext {
		visibility: hidden;
		width: auto;
		min-width: 250px;
		/*max-width: 290px;*/
		max-width: 600px;
		max-height: 469px;
		background-color: #1e1e1e;
		color: #fff;
		text-align: left;
		border-radius: 6px;
		padding: 5px 5px;
		position: absolute;
		z-index: 1;
		top: 100%;
		opacity: 0;
		transition: opacity 0.3s;
		text-overflow: ellipsis;
		white-space: normal;
		overflow-wrap: break-word;
		font-size: 100%;
		overflow-y: scroll;
		}

	.tooltip:hover .tooltiptext {
		visibility: visible;
		opacity: 1;
		}
		
	.commentTitle {
		color: #FF69B4; 
		font-weight: bold;
	}


	div.dataTables_wrapper {
		width: 100%;
	/*	margin: 0 auto;*/
	}
	.DTFC_LeftBodyLiner { overflow-x: hidden; }
	.DTFC_RightBodyLiner { overflow-x: hidden; }





</style>


</head>

	<body>

<div> <h2>test___hg19</h2></div><div id="FULL+SPLIT" class="tabcontent">
	<table id='tabFULLSPLIT' class='display compact' >
        
		<thead>
			<tr>	<th class="tooltipHeader" >AnnotSV_ID<span style="left: 1%;max-width:250px" class="tooltiptext tooltip-bottom">AnnotSV identifier</span> 	</th>
	<th class="tooltipHeader" >ACMG_class<span style="left: 71%" class="tooltiptext tooltip-bottom">SV ranking class (from 1 to 5) following the 2020 joint consensus recommendation of ACMG and ClinGen</span> 	</th>
	<th class="tooltipHeader" >SV_type<span style="left: 57%" class="tooltiptext tooltip-bottom">Type of the SV (DEL, DUP...)</span> 	</th>
	<th class="tooltipHeader" >Annotation_mode<span style="left: 42%" class="tooltiptext tooltip-bottom">Type of annotation generated:<br> - annotation on the SV full length (full)<br> - annotation on each gene overlapped by the SV (split)</span> 	</th>
	<th class="tooltipHeader" >Gene_name<span style="left: 28%" class="tooltiptext tooltip-bottom">Gene symbol</span> 	</th>
	<th class="tooltipHeader" >Location<span style="left: 14%" class="tooltiptext tooltip-bottom">SV location in the gene<br> Values: txStart, txEnd, exon#, intron# <br> e.g. txStart-exon3</span> 	</th>
	<th class="tooltipHeader" >OMIM phenotype<span style="left: 0%" class="tooltiptext tooltip-bottom">e.g. Charcot-Marie-Tooth disease</span> 	</th>
	<th class="tooltipHeader" >Exomiser score<span style="right: 14%" class="tooltiptext tooltip-bottom">Exomiser score showing how close each overlapped gene is to the patient phenotype<br> Consistent phenotype: score > 0.5<br> Highly specific and consistent phenotype: score > 0.7</span> 	</th>
	<th class="tooltipHeader" >Regulatory elements<span style="right: 28%" class="tooltiptext tooltip-bottom">Name of the genes regulated by a regulatory element overlapped with the SV to annotate.</span> 	</th>
	<th class="tooltipHeader">Benign SV sources	</th>
	<th class="tooltipHeader">Pathogenic SV sources	</th>
	<th class="tooltipHeader">Number of pathogenic SNV/indel overlapped	</th>
	<th class="tooltipHeader" >ENCODE blacklist characteristics left<span style="right: 85%" class="tooltiptext tooltip-bottom">ENCODE blacklist regions characteristics around the left SV breakpoint (+/- 100bp)</span> 	</th>
	<th class="tooltipHeader" >ENCODE blacklist characteristics right<span style="right: 100%" class="tooltiptext tooltip-bottom">ENCODE blacklist regions characteristics around the right SV breakpoint (+/- 100bp)</span> 	</th>

			</tr>
		</thead>
	<tbody>
<tr id="11_5247723_5273846_DEL_1" class="full" style="background-color:#F7D8DD" >
	<td data-order="1" data-search="11_5247723_5273846_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr11:5247723-5273846&hgt.out1=submit&highlight=hg19.chr11:5247723-5273846#aaedff" target="_blank" rel="noopener noreferrer"">11_5247723_5273846_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :<br></span> 11_5247723_5273846_DEL_1<br><span class="commentTitle">SV_length :</span> -26123<br><span class="commentTitle">Gene_count :</span> 5</span></div>	</td>
	<td data-search="5"><div class="tooltip">5<span style="left: 71%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ACMG_class :</span> 5<br><span class="commentTitle">AnnotSV_ranking_score :</span> 1.0<br><span class="commentTitle">AnnotSV_ranking_criteria :</span><br>1A: Contains protein-coding or other known functionally important elements. <br>2A (cf P_loss_source): Complete overlap of a known pathogenic Loss SV.<br>3A (5 genes): 0â€“24 genes wholly or partially included<br></span></div>	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 57%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="full"><div class="tooltip">full<span style="left: 42%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> full</span></div>	</td>
	<td style="background-color:#FFFFFF" data-search="BGLT3; HBB; HBBP1; HBD; HBG1"><div class="tooltip">BGLT3; HBB; HBBP1; HBD; HBG1<span style="left: 28%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> BGLT3; HBB; HBBP1; HBD; HBG1<br><span class="commentTitle">LOEUF_bin :</span> .<br><span class="commentTitle">GnomAD_pLI :</span> .<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 14%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> .<br><span class="commentTitle">Overlapped_tx_length :</span> .<br><span class="commentTitle">Overlapped_CDS_length :</span> .</span></div>	</td>
	<td ><div class="tooltip">.<span style="left: 0%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidates :</span> .</span></div>	</td>
	<td data-search="Absent in file"><div class="tooltip">Absent in file<span style="right: 14%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Exomiser_gene_pheno_score :</span> Absent in file</span></div>	</td>
	<td data-search="HBG2; OR51B2; OR51B4; OR51B5; OR51B6; OR51I1; OR51I2; OR51J1; OR51M1; OR51Q1; OR51V1; OR52A1; TRIM68"><div class="tooltip">HBG2; OR51B2; OR51B4; OR51B5; OR51B6; OR51I1;[...]<span style="right: 28%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> HBG2; OR51B2; OR51B4; OR51B5; OR51B6; OR51I1; OR51I2; OR51J1; OR51M1; OR51Q1; OR51V1; OR52A1; TRIM68</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 42%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td ><div class="tooltip">11:5248134-5248427; 11:5254067-5255713; 11:52[...]<span style="right: 57%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> 11:5248134-5248427; 11:5254067-5255713; 11:5269498-5271087<br><span class="commentTitle">P_loss_source :</span> CLN:581827; morbid:HBD; morbid:HBG1<br><span class="commentTitle">P_loss_phen :</span> Fetal hemoglobin quantitative trait locus 1, 141749 (3); Thalassemia due to Hb Lepore (3); Thalassemia, delta- (3); beta_Thalassemia<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td ><div class="tooltip">98<span style="right: 71%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> 98<br><span class="commentTitle">P_snvindel_phen :</span> Abnormality_of_the_hand; Anemia; Beta-knossos-thalassemia; Beta-malay-thalassemia; Beta-plus-thalassemia; Beta-thalassemia,_dominant_inclusion_body_type; Beta-thalassemia,_lermontov_type; Beta_thalassemia_intermedia; Beta_thalassemia_major; Erythrocytosis_6,_familial; Fetal_hemoglobin_quantitative_trait_locus_1; HBB-Related_Disorders; HEMOGLOBIN_C; HEMOGLOBIN_CHIBA; HEMOGLOBIN_HAMMERSMITH; HEMOGLOBIN_KNOSSOS; HEMOGLOBIN_MALAY; HEMOGLOBIN_M_(RADOM)_METHEMOGLOBINEMIA,_BETA_TYPE; HEMOGLOBIN_M_(SASKATOON); HEMOGLOBIN_OLYMPIA; HEMOGLOBIN_SABINE; HEMOGLOBIN_SANTA_ANA; HEMOGLOBIN_VILA_REAL; Hb_SS_disease; Heinz_body_anemia; Hemoglobin_E; Hemoglobin_E/beta_thalassemia_disease; Hemoglobin_E_disease; Hemoglobinopathy; Hypoplasia_of_the_ulna; Inborn_genetic_diseases; Malaria,_resistance_to; Methemoglobinemia,_beta-globin_type; Microcytic_anemia; Splenomegaly; Susceptibility_to_malaria; Thoracic_scoliosis; alpha_Thalassemia; beta^0^_Thalassemia; beta_Thalassemia; not_specified</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 85%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .<br><span class="commentTitle">ENCODE_blacklist_left :</span> .<br><span class="commentTitle">SegDup_left :</span> .<br><span class="commentTitle">Repeat_coord_left :</span> 11:5247587-5247663<br><span class="commentTitle">Repeat_type_left :</span> A-rich<br><span class="commentTitle">Gap_left :</span> .<br><span class="commentTitle">GC_content_left :</span> 0.390</span></div>	</td>
	<td data-search="Absent in file"><div class="tooltip">Absent in file<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> Absent in file<br><span class="commentTitle">ENCODE_blacklist_right :</span> .<br><span class="commentTitle">SegDup_right :</span> 11:5272859-5277141<br><span class="commentTitle">Repeat_coord_right :</span> .<br><span class="commentTitle">Repeat_type_right :</span> .<br><span class="commentTitle">Gap_right :</span> .<br><span class="commentTitle">GC_content_right :</span> 0.415</span></div>	</td>
</tr>
<tr class="fullsplit 11_5247723_5273846_DEL_1" >
	<td data-order="2" data-search="11_5247723_5273846_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr11:5247723-5273846&hgt.out1=submit&highlight=hg19.chr11:5247723-5273846#aaedff" target="_blank" rel="noopener noreferrer"">11_5247723_5273846_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :<br></span> 11_5247723_5273846_DEL_1<br><span class="commentTitle">SV_length :</span> -26123</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 71%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ACMG_class :</span> .</span></div>	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 57%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 42%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#BFFF00" data-search="HBG1"><div class="tooltip"><a href="https://www.genenames.org/tools/search/#!/all?query=HBG1" target="_blank" rel="noopener noreferrer"">HBG1</a><span style="left: 28%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> HBG1<br><span class="commentTitle">LOEUF_bin :</span> 6<br><span class="commentTitle">GnomAD_pLI :</span> 5.6611e-01<br><span class="commentTitle">ExAC_pLI :</span> 2.9779e-02<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 86.01<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> 1.6907442803648<br><span class="commentTitle">ExAC_misZ :</span> 1.62238135952892<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 14%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_000559<br><span class="commentTitle">Exon_count :</span> 3<br><span class="commentTitle">Overlapped_tx_length :</span> 1589<br><span class="commentTitle">Overlapped_CDS_length :</span> 444<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_Nearest_SS :</span> .<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_Start :</span> 5269498<br><span class="commentTitle">Intersect_End :</span> 5271087</span></div>	</td>
	<td ><div class="tooltip">Fetal hemoglobin quantitative trait locus 1, [...]<span style="left: 0%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> Fetal hemoglobin quantitative trait locus 1, 141749 (3)<br><span class="commentTitle">OMIM_ID :</span> <a href="https://www.omim.org/entry/142200" target="_blank" rel="noopener noreferrer" style="color:white">142200</a><br><span class="commentTitle">OMIM_inheritance :</span> AD<br><span class="commentTitle">OMIM_morbid :</span> yes<br><span class="commentTitle">OMIM_morbid_candidates :</span> .</span></div>	</td>
	<td data-search="Absent in file"><div class="tooltip">Absent in file<span style="right: 14%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Exomiser_gene_pheno_score :</span> Absent in file<br><span class="commentTitle">Human_pheno_evidence :</span> Absent in file<br><span class="commentTitle">Mouse_pheno_evidence :</span> Absent in file<br><span class="commentTitle">Fish_pheno_evidence :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 28%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 42%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td ><div class="tooltip">11:5269498-5271087<span style="right: 57%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> 11:5269498-5271087<br><span class="commentTitle">P_loss_source :</span> morbid:HBG1<br><span class="commentTitle">P_loss_phen :</span> Fetal hemoglobin quantitative trait locus 1, 141749 (3)<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td ><div class="tooltip">.<span style="right: 71%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 85%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="Absent in file"><div class="tooltip">Absent in file<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> Absent in file</span></div>	</td>
</tr>
<tr class="fullsplit 11_5247723_5273846_DEL_1" >
	<td data-order="3" data-search="11_5247723_5273846_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr11:5247723-5273846&hgt.out1=submit&highlight=hg19.chr11:5247723-5273846#aaedff" target="_blank" rel="noopener noreferrer"">11_5247723_5273846_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :<br></span> 11_5247723_5273846_DEL_1<br><span class="commentTitle">SV_length :</span> -26123</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 71%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ACMG_class :</span> .</span></div>	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 57%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 42%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background: linear-gradient(-45deg, white 50%, #00FF00 50% )" data-search="HBB"><div class="tooltip"><a href="https://www.genenames.org/tools/search/#!/all?query=HBB" target="_blank" rel="noopener noreferrer"">HBB</a><span style="left: 28%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> HBB<br><span class="commentTitle">LOEUF_bin :</span> 9<br><span class="commentTitle">GnomAD_pLI :</span> 1.2274e-09<br><span class="commentTitle">ExAC_pLI :</span> 1.0249e-08<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 57.13<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> -2.47162789907313<br><span class="commentTitle">ExAC_delZ :</span> -2.62354415217069<br><span class="commentTitle">ExAC_dupZ :</span> -2.53125239294469<br><span class="commentTitle">ExAC_synZ :</span> -1.96331260869213<br><span class="commentTitle">ExAC_misZ :</span> -0.568017270792556<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-intron2"><div class="tooltip">txStart-intron2<span style="left: 14%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-intron2<br><span class="commentTitle">Location2 :</span> 5'UTR-CDS<br><span class="commentTitle">Tx :</span> NM_000518<br><span class="commentTitle">Exon_count :</span> 3<br><span class="commentTitle">Overlapped_tx_length :</span> 578<br><span class="commentTitle">Overlapped_CDS_length :</span> 315<br><span class="commentTitle">Overlapped_CDS_percent :</span> 70<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_Nearest_SS :</span> 83<br><span class="commentTitle">Nearest_SS_type :</span> 5'<br><span class="commentTitle">Intersect_Start :</span> 5247723<br><span class="commentTitle">Intersect_End :</span> 5248301</span></div>	</td>
	<td ><div class="tooltip">Thalassemia, beta, 613985 (3)/ Erythrocytosis[...]<span style="left: 0%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> Thalassemia, beta, 613985 (3)/ Erythrocytosis 6, 617980 (3)/ Heinz body anemia, 140700 (3)/ Delta-beta thalassemia, 141749 (3)/ Thalassemia-beta, dominant inclusion-body, 603902 (3)/ Hereditary persistence of fetal hemoglobin, 141749 (3)/ Methemoglobinemia, beta type, 617971 (3)/ (Malaria, resistance to), 611162 (3)/ Sickle cell anemia, 603903 (3)<br><span class="commentTitle">OMIM_ID :</span> <a href="https://www.omim.org/entry/141900" target="_blank" rel="noopener noreferrer" style="color:white">141900</a><br><span class="commentTitle">OMIM_inheritance :</span> //AD/AD//AD///AR<br><span class="commentTitle">OMIM_morbid :</span> yes<br><span class="commentTitle">OMIM_morbid_candidates :</span> yes</span></div>	</td>
	<td data-search="Absent in file"><div class="tooltip">Absent in file<span style="right: 14%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Exomiser_gene_pheno_score :</span> Absent in file<br><span class="commentTitle">Human_pheno_evidence :</span> Absent in file<br><span class="commentTitle">Mouse_pheno_evidence :</span> Absent in file<br><span class="commentTitle">Fish_pheno_evidence :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 28%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 42%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td ><div class="tooltip">.<span style="right: 57%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td ><div class="tooltip">82<span style="right: 71%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> 82<br><span class="commentTitle">P_snvindel_phen :</span> Abnormality_of_the_hand; Anemia; Beta-knossos-thalassemia; Beta-malay-thalassemia; Beta-plus-thalassemia; Beta-thalassemia,_dominant_inclusion_body_type; Beta-thalassemia,_lermontov_type; Beta_thalassemia_major; Erythrocytosis_6,_familial; Fetal_hemoglobin_quantitative_trait_locus_1; HBB-Related_Disorders; HEMOGLOBIN_C; HEMOGLOBIN_CHIBA; HEMOGLOBIN_HAMMERSMITH; HEMOGLOBIN_KNOSSOS; HEMOGLOBIN_MALAY; HEMOGLOBIN_M_(RADOM)_METHEMOGLOBINEMIA,_BETA_TYPE; HEMOGLOBIN_M_(SASKATOON); HEMOGLOBIN_OLYMPIA; HEMOGLOBIN_SABINE; HEMOGLOBIN_SANTA_ANA; HEMOGLOBIN_VILA_REAL; Hb_SS_disease; Heinz_body_anemia; Hemoglobin_E; Hemoglobin_E/beta_thalassemia_disease; Hemoglobin_E_disease; Hemoglobinopathy; Hypoplasia_of_the_ulna; Inborn_genetic_diseases; Malaria,_resistance_to; Methemoglobinemia,_beta-globin_type; Microcytic_anemia; Splenomegaly; Susceptibility_to_malaria; Thoracic_scoliosis; alpha_Thalassemia; beta^0^_Thalassemia; beta_Thalassemia; not_specified</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 85%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="Absent in file"><div class="tooltip">Absent in file<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> Absent in file</span></div>	</td>
</tr>
<tr class="fullsplit 11_5247723_5273846_DEL_1" >
	<td data-order="4" data-search="11_5247723_5273846_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr11:5247723-5273846&hgt.out1=submit&highlight=hg19.chr11:5247723-5273846#aaedff" target="_blank" rel="noopener noreferrer"">11_5247723_5273846_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :<br></span> 11_5247723_5273846_DEL_1<br><span class="commentTitle">SV_length :</span> -26123</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 71%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ACMG_class :</span> .</span></div>	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 57%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 42%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFFFF" data-search="HBBP1"><div class="tooltip"><a href="https://www.genenames.org/tools/search/#!/all?query=HBBP1" target="_blank" rel="noopener noreferrer"">HBBP1</a><span style="left: 28%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> HBBP1<br><span class="commentTitle">LOEUF_bin :</span> .<br><span class="commentTitle">GnomAD_pLI :</span> .<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> .<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 14%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> UTR<br><span class="commentTitle">Tx :</span> NR_001589<br><span class="commentTitle">Exon_count :</span> 3<br><span class="commentTitle">Overlapped_tx_length :</span> 1638<br><span class="commentTitle">Overlapped_CDS_length :</span> 0<br><span class="commentTitle">Overlapped_CDS_percent :</span> 0<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_Nearest_SS :</span> .<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_Start :</span> 5263184<br><span class="commentTitle">Intersect_End :</span> 5264822</span></div>	</td>
	<td ><div class="tooltip">.<span style="left: 0%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidates :</span> .</span></div>	</td>
	<td data-search="Absent in file"><div class="tooltip">Absent in file<span style="right: 14%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Exomiser_gene_pheno_score :</span> Absent in file<br><span class="commentTitle">Human_pheno_evidence :</span> Absent in file<br><span class="commentTitle">Mouse_pheno_evidence :</span> Absent in file<br><span class="commentTitle">Fish_pheno_evidence :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 28%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 42%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td ><div class="tooltip">.<span style="right: 57%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td ><div class="tooltip">.<span style="right: 71%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 85%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="Absent in file"><div class="tooltip">Absent in file<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> Absent in file</span></div>	</td>
</tr>
<tr class="fullsplit 11_5247723_5273846_DEL_1" >
	<td data-order="5" data-search="11_5247723_5273846_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr11:5247723-5273846&hgt.out1=submit&highlight=hg19.chr11:5247723-5273846#aaedff" target="_blank" rel="noopener noreferrer"">11_5247723_5273846_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :<br></span> 11_5247723_5273846_DEL_1<br><span class="commentTitle">SV_length :</span> -26123</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 71%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ACMG_class :</span> .</span></div>	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 57%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 42%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#00FF00" data-search="HBD"><div class="tooltip"><a href="https://www.genenames.org/tools/search/#!/all?query=HBD" target="_blank" rel="noopener noreferrer"">HBD</a><span style="left: 28%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> HBD<br><span class="commentTitle">LOEUF_bin :</span> 9<br><span class="commentTitle">GnomAD_pLI :</span> 1.9236e-05<br><span class="commentTitle">ExAC_pLI :</span> 1.5511e-04<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 77.18<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> -0.203398233874436<br><span class="commentTitle">ExAC_misZ :</span> -1.15614462192474<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 14%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_000519<br><span class="commentTitle">Exon_count :</span> 3<br><span class="commentTitle">Overlapped_tx_length :</span> 1646<br><span class="commentTitle">Overlapped_CDS_length :</span> 444<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_Nearest_SS :</span> .<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_Start :</span> 5254067<br><span class="commentTitle">Intersect_End :</span> 5255713</span></div>	</td>
	<td ><div class="tooltip">Thalassemia, delta- (3)/ Thalassemia due to H[...]<span style="left: 0%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> Thalassemia, delta- (3)/ Thalassemia due to Hb Lepore (3)<br><span class="commentTitle">OMIM_ID :</span> <a href="https://www.omim.org/entry/142000" target="_blank" rel="noopener noreferrer" style="color:white">142000</a><br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> yes<br><span class="commentTitle">OMIM_morbid_candidates :</span> .</span></div>	</td>
	<td data-search="Absent in file"><div class="tooltip">Absent in file<span style="right: 14%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Exomiser_gene_pheno_score :</span> Absent in file<br><span class="commentTitle">Human_pheno_evidence :</span> Absent in file<br><span class="commentTitle">Mouse_pheno_evidence :</span> Absent in file<br><span class="commentTitle">Fish_pheno_evidence :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 28%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 42%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td ><div class="tooltip">11:5254067-5255713<span style="right: 57%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> 11:5254067-5255713<br><span class="commentTitle">P_loss_source :</span> morbid:HBD<br><span class="commentTitle">P_loss_phen :</span> Thalassemia due to Hb Lepore (3); Thalassemia, delta- (3)<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td ><div class="tooltip">.<span style="right: 71%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 85%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="Absent in file"><div class="tooltip">Absent in file<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> Absent in file</span></div>	</td>
</tr>
<tr class="fullsplit 11_5247723_5273846_DEL_1" >
	<td data-order="6" data-search="11_5247723_5273846_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr11:5247723-5273846&hgt.out1=submit&highlight=hg19.chr11:5247723-5273846#aaedff" target="_blank" rel="noopener noreferrer"">11_5247723_5273846_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :<br></span> 11_5247723_5273846_DEL_1<br><span class="commentTitle">SV_length :</span> -26123</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 71%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ACMG_class :</span> .</span></div>	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 57%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 42%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFFFF" data-search="BGLT3"><div class="tooltip"><a href="https://www.genenames.org/tools/search/#!/all?query=BGLT3" target="_blank" rel="noopener noreferrer"">BGLT3</a><span style="left: 28%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> BGLT3<br><span class="commentTitle">LOEUF_bin :</span> .<br><span class="commentTitle">GnomAD_pLI :</span> .<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> .<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 14%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> UTR<br><span class="commentTitle">Tx :</span> NR_121648<br><span class="commentTitle">Exon_count :</span> 1<br><span class="commentTitle">Overlapped_tx_length :</span> 993<br><span class="commentTitle">Overlapped_CDS_length :</span> 0<br><span class="commentTitle">Overlapped_CDS_percent :</span> 0<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_Nearest_SS :</span> .<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_Start :</span> 5265783<br><span class="commentTitle">Intersect_End :</span> 5266776</span></div>	</td>
	<td ><div class="tooltip">.<span style="left: 0%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidates :</span> .</span></div>	</td>
	<td data-search="Absent in file"><div class="tooltip">Absent in file<span style="right: 14%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Exomiser_gene_pheno_score :</span> Absent in file<br><span class="commentTitle">Human_pheno_evidence :</span> Absent in file<br><span class="commentTitle">Mouse_pheno_evidence :</span> Absent in file<br><span class="commentTitle">Fish_pheno_evidence :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 28%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 42%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td ><div class="tooltip">.<span style="right: 57%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td ><div class="tooltip">.<span style="right: 71%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 85%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="Absent in file"><div class="tooltip">Absent in file<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> Absent in file</span></div>	</td>
</tr>
<tr id="2_142216368_148540744_DEL_1" class="full" style="background-color:#F7D8DD" >
	<td data-order="7" data-search="2_142216368_148540744_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr2:142216368-148540744&hgt.out1=submit&highlight=hg19.chr2:142216368-148540744#aaedff" target="_blank" rel="noopener noreferrer"">2_142216368_148540744_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :<br></span> 2_142216368_148540744_DEL_1<br><span class="commentTitle">SV_length :</span> -6324376<br><span class="commentTitle">Gene_count :</span> 11</span></div>	</td>
	<td data-search="5"><div class="tooltip">5<span style="left: 71%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ACMG_class :</span> 5<br><span class="commentTitle">AnnotSV_ranking_score :</span> 1.0<br><span class="commentTitle">AnnotSV_ranking_criteria :</span><br>1A: Contains protein-coding or other known functionally important elements. <br>2A (cf P_loss_source): Complete overlap of a known pathogenic Loss SV.<br>3A (11 genes): 0â€“24 genes wholly or partially included<br></span></div>	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 57%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="full"><div class="tooltip">full<span style="left: 42%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> full</span></div>	</td>
	<td style="background-color:#FFFFFF" data-search="ARHGAP15; GTDC1; KYNU; LINC01412; LOC101928386; LOC105373656; LRP1B; PABPC1P2; TEX41; ZEB2; ZEB2-AS1"><div class="tooltip">ARHGAP15; GTDC1; KYNU; LINC01412; LOC10192838[...]<span style="left: 28%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> ARHGAP15; GTDC1; KYNU; LINC01412; LOC101928386; LOC105373656; LRP1B; PABPC1P2; TEX41; ZEB2; ZEB2-AS1<br><span class="commentTitle">LOEUF_bin :</span> .<br><span class="commentTitle">GnomAD_pLI :</span> .<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 14%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> .<br><span class="commentTitle">Overlapped_tx_length :</span> .<br><span class="commentTitle">Overlapped_CDS_length :</span> .</span></div>	</td>
	<td ><div class="tooltip">.<span style="left: 0%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidates :</span> .</span></div>	</td>
	<td data-search="Absent in file"><div class="tooltip">Absent in file<span style="right: 14%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Exomiser_gene_pheno_score :</span> Absent in file</span></div>	</td>
	<td data-search="ACVR2A; MBD5 (HI=3; TS=1); ORC4 (HI=30; TS=0)"><div class="tooltip">ACVR2A; MBD5 (HI=3; TS=1); ORC4 (HI=30; TS=0)<span style="right: 28%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> ACVR2A; MBD5 (HI=3; TS=1); ORC4 (HI=30; TS=0)</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 42%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td ><div class="tooltip">2:143634576-143719262; 2:143635232-143813402;[...]<span style="right: 57%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> 2:143634576-143719262; 2:143635232-143813402; 2:145141942-145277958; 2:145268951-145277686<br><span class="commentTitle">P_loss_source :</span> CLN:966395; HI3:ZEB2; morbid:KYNU; morbid:ZEB2<br><span class="commentTitle">P_loss_phen :</span> ?Hydroxykynureninuria, 236800 (3); Catel_Manzke_syndrome; Mowat-Wilson syndrome, 235730 (3); Vertebral, cardiac, renal, and limb defects syndrome 2, 617661 (3)<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td ><div class="tooltip">135<span style="right: 71%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> 135<br><span class="commentTitle">P_snvindel_phen :</span> Catel_Manzke_syndrome; Congenital_NAD_deficiency_disorder; History_of_neurodevelopmental_disorder; Inborn_genetic_diseases; Intellectual_disability; Mowat-Wilson_syndrome; Smith-Magenis_Syndrome-like; VERTEBRAL,_CARDIAC,_RENAL,_AND_LIMB_DEFECTS_SYNDROME_2</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 85%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .<br><span class="commentTitle">ENCODE_blacklist_left :</span> .<br><span class="commentTitle">SegDup_left :</span> .<br><span class="commentTitle">Repeat_coord_left :</span> .<br><span class="commentTitle">Repeat_type_left :</span> .<br><span class="commentTitle">Gap_left :</span> .<br><span class="commentTitle">GC_content_left :</span> 0.320</span></div>	</td>
	<td data-search="Absent in file"><div class="tooltip">Absent in file<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> Absent in file<br><span class="commentTitle">ENCODE_blacklist_right :</span> .<br><span class="commentTitle">SegDup_right :</span> .<br><span class="commentTitle">Repeat_coord_right :</span> .<br><span class="commentTitle">Repeat_type_right :</span> .<br><span class="commentTitle">Gap_right :</span> .<br><span class="commentTitle">GC_content_right :</span> 0.340</span></div>	</td>
</tr>
<tr class="fullsplit 2_142216368_148540744_DEL_1" >
	<td data-order="8" data-search="2_142216368_148540744_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr2:142216368-148540744&hgt.out1=submit&highlight=hg19.chr2:142216368-148540744#aaedff" target="_blank" rel="noopener noreferrer"">2_142216368_148540744_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :<br></span> 2_142216368_148540744_DEL_1<br><span class="commentTitle">SV_length :</span> -6324376</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 71%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ACMG_class :</span> .</span></div>	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 57%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 42%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background: linear-gradient(-45deg, white 50%, #FF3300 50% )" data-search="LRP1B"><div class="tooltip"><a href="https://www.genenames.org/tools/search/#!/all?query=LRP1B" target="_blank" rel="noopener noreferrer"">LRP1B</a><span style="left: 28%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> LRP1B<br><span class="commentTitle">LOEUF_bin :</span> 1<br><span class="commentTitle">GnomAD_pLI :</span> 2.5267e-06<br><span class="commentTitle">ExAC_pLI :</span> 1.0000e+00<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 1.62<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> 0.329622238840903<br><span class="commentTitle">ExAC_delZ :</span> -0.672226653195572<br><span class="commentTitle">ExAC_dupZ :</span> 0.849209882795508<br><span class="commentTitle">ExAC_synZ :</span> -1.57118316036647<br><span class="commentTitle">ExAC_misZ :</span> 0.672316636355286<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-intron3"><div class="tooltip">txStart-intron3<span style="left: 14%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-intron3<br><span class="commentTitle">Location2 :</span> 5'UTR-CDS<br><span class="commentTitle">Tx :</span> NM_018557<br><span class="commentTitle">Exon_count :</span> 91<br><span class="commentTitle">Overlapped_tx_length :</span> 672217<br><span class="commentTitle">Overlapped_CDS_length :</span> 343<br><span class="commentTitle">Overlapped_CDS_percent :</span> 2<br><span class="commentTitle">Frameshift :</span> yes<br><span class="commentTitle">Dist_Nearest_SS :</span> 21596<br><span class="commentTitle">Nearest_SS_type :</span> 5'<br><span class="commentTitle">Intersect_Start :</span> 142216368<br><span class="commentTitle">Intersect_End :</span> 142888585</span></div>	</td>
	<td ><div class="tooltip">.<span style="left: 0%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidates :</span> .</span></div>	</td>
	<td data-search="Absent in file"><div class="tooltip">Absent in file<span style="right: 14%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Exomiser_gene_pheno_score :</span> Absent in file<br><span class="commentTitle">Human_pheno_evidence :</span> Absent in file<br><span class="commentTitle">Mouse_pheno_evidence :</span> Absent in file<br><span class="commentTitle">Fish_pheno_evidence :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 28%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 42%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td ><div class="tooltip">.<span style="right: 57%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td ><div class="tooltip">.<span style="right: 71%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 85%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="Absent in file"><div class="tooltip">Absent in file<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> Absent in file</span></div>	</td>
</tr>
<tr class="fullsplit 2_142216368_148540744_DEL_1" >
	<td data-order="9" data-search="2_142216368_148540744_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr2:142216368-148540744&hgt.out1=submit&highlight=hg19.chr2:142216368-148540744#aaedff" target="_blank" rel="noopener noreferrer"">2_142216368_148540744_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :<br></span> 2_142216368_148540744_DEL_1<br><span class="commentTitle">SV_length :</span> -6324376</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 71%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ACMG_class :</span> .</span></div>	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 57%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 42%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFCC00" data-search="ARHGAP15"><div class="tooltip"><a href="https://www.genenames.org/tools/search/#!/all?query=ARHGAP15" target="_blank" rel="noopener noreferrer"">ARHGAP15</a><span style="left: 28%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> ARHGAP15<br><span class="commentTitle">LOEUF_bin :</span> 4<br><span class="commentTitle">GnomAD_pLI :</span> 1.3720e-08<br><span class="commentTitle">ExAC_pLI :</span> 5.2541e-06<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 0.39<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> -0.162050293981424<br><span class="commentTitle">ExAC_delZ :</span> 0.0693083894648098<br><span class="commentTitle">ExAC_dupZ :</span> -0.313560681801348<br><span class="commentTitle">ExAC_synZ :</span> -0.414674347136382<br><span class="commentTitle">ExAC_misZ :</span> -0.314567510411296<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 14%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_018460<br><span class="commentTitle">Exon_count :</span> 14<br><span class="commentTitle">Overlapped_tx_length :</span> 638934<br><span class="commentTitle">Overlapped_CDS_length :</span> 1428<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_Nearest_SS :</span> .<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_Start :</span> 143886987<br><span class="commentTitle">Intersect_End :</span> 144525921</span></div>	</td>
	<td ><div class="tooltip">.<span style="left: 0%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidates :</span> .</span></div>	</td>
	<td data-search="Absent in file"><div class="tooltip">Absent in file<span style="right: 14%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Exomiser_gene_pheno_score :</span> Absent in file<br><span class="commentTitle">Human_pheno_evidence :</span> Absent in file<br><span class="commentTitle">Mouse_pheno_evidence :</span> Absent in file<br><span class="commentTitle">Fish_pheno_evidence :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 28%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 42%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td ><div class="tooltip">.<span style="right: 57%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td ><div class="tooltip">.<span style="right: 71%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 85%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="Absent in file"><div class="tooltip">Absent in file<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> Absent in file</span></div>	</td>
</tr>
<tr class="fullsplit 2_142216368_148540744_DEL_1" >
	<td data-order="10" data-search="2_142216368_148540744_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr2:142216368-148540744&hgt.out1=submit&highlight=hg19.chr2:142216368-148540744#aaedff" target="_blank" rel="noopener noreferrer"">2_142216368_148540744_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :<br></span> 2_142216368_148540744_DEL_1<br><span class="commentTitle">SV_length :</span> -6324376</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 71%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ACMG_class :</span> .</span></div>	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 57%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 42%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#7FFF00" data-search="GTDC1"><div class="tooltip"><a href="https://www.genenames.org/tools/search/#!/all?query=GTDC1" target="_blank" rel="noopener noreferrer"">GTDC1</a><span style="left: 28%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> GTDC1<br><span class="commentTitle">LOEUF_bin :</span> 7<br><span class="commentTitle">GnomAD_pLI :</span> 1.0127e-12<br><span class="commentTitle">ExAC_pLI :</span> 2.4090e-10<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 2.92<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> -0.411459590620902<br><span class="commentTitle">ExAC_delZ :</span> -1.32758543319309<br><span class="commentTitle">ExAC_dupZ :</span> 0.276107196209042<br><span class="commentTitle">ExAC_synZ :</span> 0.90437703811114<br><span class="commentTitle">ExAC_misZ :</span> -1.23824868015629<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 14%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_001376306<br><span class="commentTitle">Exon_count :</span> 13<br><span class="commentTitle">Overlapped_tx_length :</span> 394443<br><span class="commentTitle">Overlapped_CDS_length :</span> 1524<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_Nearest_SS :</span> .<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_Start :</span> 144695636<br><span class="commentTitle">Intersect_End :</span> 145090079</span></div>	</td>
	<td ><div class="tooltip">.<span style="left: 0%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidates :</span> .</span></div>	</td>
	<td data-search="Absent in file"><div class="tooltip">Absent in file<span style="right: 14%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Exomiser_gene_pheno_score :</span> Absent in file<br><span class="commentTitle">Human_pheno_evidence :</span> Absent in file<br><span class="commentTitle">Mouse_pheno_evidence :</span> Absent in file<br><span class="commentTitle">Fish_pheno_evidence :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 28%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 42%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td ><div class="tooltip">.<span style="right: 57%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td ><div class="tooltip">.<span style="right: 71%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 85%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="Absent in file"><div class="tooltip">Absent in file<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> Absent in file</span></div>	</td>
</tr>
<tr class="fullsplit 2_142216368_148540744_DEL_1" >
	<td data-order="11" data-search="2_142216368_148540744_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr2:142216368-148540744&hgt.out1=submit&highlight=hg19.chr2:142216368-148540744#aaedff" target="_blank" rel="noopener noreferrer"">2_142216368_148540744_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :<br></span> 2_142216368_148540744_DEL_1<br><span class="commentTitle">SV_length :</span> -6324376</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 71%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ACMG_class :</span> .</span></div>	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 57%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 42%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#7FFF00" data-search="KYNU"><div class="tooltip"><a href="https://www.genenames.org/tools/search/#!/all?query=KYNU" target="_blank" rel="noopener noreferrer"">KYNU</a><span style="left: 28%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> KYNU<br><span class="commentTitle">LOEUF_bin :</span> 7<br><span class="commentTitle">GnomAD_pLI :</span> 4.1806e-18<br><span class="commentTitle">ExAC_pLI :</span> 1.8277e-09<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 41.86<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> 0.378525104033653<br><span class="commentTitle">ExAC_delZ :</span> 0.00626463044787885<br><span class="commentTitle">ExAC_dupZ :</span> 0.495546633769546<br><span class="commentTitle">ExAC_synZ :</span> -0.493020841165644<br><span class="commentTitle">ExAC_misZ :</span> -1.14319607793643<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 14%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_001199241<br><span class="commentTitle">Exon_count :</span> 15<br><span class="commentTitle">Overlapped_tx_length :</span> 178170<br><span class="commentTitle">Overlapped_CDS_length :</span> 1398<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_Nearest_SS :</span> .<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_Start :</span> 143635232<br><span class="commentTitle">Intersect_End :</span> 143813402</span></div>	</td>
	<td ><div class="tooltip">Vertebral, cardiac, renal, and limb defects s[...]<span style="left: 0%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> Vertebral, cardiac, renal, and limb defects syndrome 2, 617661 (3)/ ?Hydroxykynureninuria, 236800 (3)<br><span class="commentTitle">OMIM_ID :</span> <a href="https://www.omim.org/entry/605197" target="_blank" rel="noopener noreferrer" style="color:white">605197</a><br><span class="commentTitle">OMIM_inheritance :</span> AR<br><span class="commentTitle">OMIM_morbid :</span> yes<br><span class="commentTitle">OMIM_morbid_candidates :</span> yes</span></div>	</td>
	<td data-search="Absent in file"><div class="tooltip">Absent in file<span style="right: 14%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Exomiser_gene_pheno_score :</span> Absent in file<br><span class="commentTitle">Human_pheno_evidence :</span> Absent in file<br><span class="commentTitle">Mouse_pheno_evidence :</span> Absent in file<br><span class="commentTitle">Fish_pheno_evidence :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 28%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 42%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td ><div class="tooltip">2:143635232-143813402<span style="right: 57%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> 2:143635232-143813402<br><span class="commentTitle">P_loss_source :</span> morbid:KYNU<br><span class="commentTitle">P_loss_phen :</span> ?Hydroxykynureninuria, 236800 (3); Vertebral, cardiac, renal, and limb defects syndrome 2, 617661 (3)<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td ><div class="tooltip">6<span style="right: 71%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> 6<br><span class="commentTitle">P_snvindel_phen :</span> Catel_Manzke_syndrome; Congenital_NAD_deficiency_disorder; VERTEBRAL,_CARDIAC,_RENAL,_AND_LIMB_DEFECTS_SYNDROME_2</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 85%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="Absent in file"><div class="tooltip">Absent in file<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> Absent in file</span></div>	</td>
</tr>
<tr class="fullsplit 2_142216368_148540744_DEL_1" >
	<td data-order="12" data-search="2_142216368_148540744_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr2:142216368-148540744&hgt.out1=submit&highlight=hg19.chr2:142216368-148540744#aaedff" target="_blank" rel="noopener noreferrer"">2_142216368_148540744_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :<br></span> 2_142216368_148540744_DEL_1<br><span class="commentTitle">SV_length :</span> -6324376</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 71%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ACMG_class :</span> .</span></div>	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 57%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 42%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FF0000" data-search="ZEB2"><div class="tooltip"><a href="https://www.genenames.org/tools/search/#!/all?query=ZEB2" target="_blank" rel="noopener noreferrer"">ZEB2</a><span style="left: 28%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> ZEB2<br><span class="commentTitle">LOEUF_bin :</span> 0<br><span class="commentTitle">GnomAD_pLI :</span> 1.0000e+00<br><span class="commentTitle">ExAC_pLI :</span> 9.9983e-01<br><span class="commentTitle">HI :</span> 3<br><span class="commentTitle">TS :</span> 0<br><span class="commentTitle">DDD_HI_percent :</span> 0.37<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> -0.631528877179671<br><span class="commentTitle">ExAC_delZ :</span> -0.949172238522553<br><span class="commentTitle">ExAC_dupZ :</span> -0.414940535299876<br><span class="commentTitle">ExAC_synZ :</span> 1.20087572946893<br><span class="commentTitle">ExAC_misZ :</span> 4.94560342770108<br><span class="commentTitle">DDD_status :</span> confirmed<br><span class="commentTitle">DDD_mode :</span> monoallelic<br><span class="commentTitle">DDD_consequence :</span> loss of function<br><span class="commentTitle">DDD_disease :</span> MOWAT-WILSON SYNDROME<br><span class="commentTitle">DDD_pmid :</span> 9719364; 16532472; 16688751; 11279515; 11891681; 12920073; 12451214; 11592033; 16088920; 11595972</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 14%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_014795<br><span class="commentTitle">Exon_count :</span> 10<br><span class="commentTitle">Overlapped_tx_length :</span> 136039<br><span class="commentTitle">Overlapped_CDS_length :</span> 3645<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_Nearest_SS :</span> .<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_Start :</span> 145141647<br><span class="commentTitle">Intersect_End :</span> 145277686</span></div>	</td>
	<td ><div class="tooltip">Mowat-Wilson syndrome, 235730 (3)<span style="left: 0%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> Mowat-Wilson syndrome, 235730 (3)<br><span class="commentTitle">OMIM_ID :</span> <a href="https://www.omim.org/entry/605802" target="_blank" rel="noopener noreferrer" style="color:white">605802</a><br><span class="commentTitle">OMIM_inheritance :</span> AD<br><span class="commentTitle">OMIM_morbid :</span> yes<br><span class="commentTitle">OMIM_morbid_candidates :</span> .</span></div>	</td>
	<td data-search="Absent in file"><div class="tooltip">Absent in file<span style="right: 14%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Exomiser_gene_pheno_score :</span> Absent in file<br><span class="commentTitle">Human_pheno_evidence :</span> Absent in file<br><span class="commentTitle">Mouse_pheno_evidence :</span> Absent in file<br><span class="commentTitle">Fish_pheno_evidence :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 28%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 42%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td ><div class="tooltip">2:145268951-145277686<span style="right: 57%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> 2:145268951-145277686<br><span class="commentTitle">P_loss_source :</span> morbid:ZEB2<br><span class="commentTitle">P_loss_phen :</span> Mowat-Wilson syndrome, 235730 (3)<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td ><div class="tooltip">129<span style="right: 71%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> 129<br><span class="commentTitle">P_snvindel_phen :</span> History_of_neurodevelopmental_disorder; Inborn_genetic_diseases; Intellectual_disability; Mowat-Wilson_syndrome; Smith-Magenis_Syndrome-like</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 85%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="Absent in file"><div class="tooltip">Absent in file<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> Absent in file</span></div>	</td>
</tr>
<tr class="fullsplit 2_142216368_148540744_DEL_1" >
	<td data-order="13" data-search="2_142216368_148540744_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr2:142216368-148540744&hgt.out1=submit&highlight=hg19.chr2:142216368-148540744#aaedff" target="_blank" rel="noopener noreferrer"">2_142216368_148540744_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :<br></span> 2_142216368_148540744_DEL_1<br><span class="commentTitle">SV_length :</span> -6324376</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 71%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ACMG_class :</span> .</span></div>	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 57%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 42%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFFFF" data-search="TEX41"><div class="tooltip"><a href="https://www.genenames.org/tools/search/#!/all?query=TEX41" target="_blank" rel="noopener noreferrer"">TEX41</a><span style="left: 28%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> TEX41<br><span class="commentTitle">LOEUF_bin :</span> .<br><span class="commentTitle">GnomAD_pLI :</span> .<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> .<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 14%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> UTR<br><span class="commentTitle">Tx :</span> NR_033870<br><span class="commentTitle">Exon_count :</span> 5<br><span class="commentTitle">Overlapped_tx_length :</span> 408758<br><span class="commentTitle">Overlapped_CDS_length :</span> 0<br><span class="commentTitle">Overlapped_CDS_percent :</span> 0<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_Nearest_SS :</span> .<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_Start :</span> 145425533<br><span class="commentTitle">Intersect_End :</span> 145834291</span></div>	</td>
	<td ><div class="tooltip">.<span style="left: 0%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidates :</span> .</span></div>	</td>
	<td data-search="Absent in file"><div class="tooltip">Absent in file<span style="right: 14%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Exomiser_gene_pheno_score :</span> Absent in file<br><span class="commentTitle">Human_pheno_evidence :</span> Absent in file<br><span class="commentTitle">Mouse_pheno_evidence :</span> Absent in file<br><span class="commentTitle">Fish_pheno_evidence :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 28%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 42%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td ><div class="tooltip">.<span style="right: 57%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td ><div class="tooltip">.<span style="right: 71%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 85%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="Absent in file"><div class="tooltip">Absent in file<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> Absent in file</span></div>	</td>
</tr>
<tr class="fullsplit 2_142216368_148540744_DEL_1" >
	<td data-order="14" data-search="2_142216368_148540744_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr2:142216368-148540744&hgt.out1=submit&highlight=hg19.chr2:142216368-148540744#aaedff" target="_blank" rel="noopener noreferrer"">2_142216368_148540744_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :<br></span> 2_142216368_148540744_DEL_1<br><span class="commentTitle">SV_length :</span> -6324376</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 71%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ACMG_class :</span> .</span></div>	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 57%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 42%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFFFF" data-search="ZEB2-AS1"><div class="tooltip"><a href="https://www.genenames.org/tools/search/#!/all?query=ZEB2-AS1" target="_blank" rel="noopener noreferrer"">ZEB2-AS1</a><span style="left: 28%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> ZEB2-AS1<br><span class="commentTitle">LOEUF_bin :</span> .<br><span class="commentTitle">GnomAD_pLI :</span> .<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> .<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 14%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> UTR<br><span class="commentTitle">Tx :</span> NR_040248<br><span class="commentTitle">Exon_count :</span> 4<br><span class="commentTitle">Overlapped_tx_length :</span> 1285<br><span class="commentTitle">Overlapped_CDS_length :</span> 0<br><span class="commentTitle">Overlapped_CDS_percent :</span> 0<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_Nearest_SS :</span> .<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_Start :</span> 145277180<br><span class="commentTitle">Intersect_End :</span> 145278465</span></div>	</td>
	<td ><div class="tooltip">.<span style="left: 0%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidates :</span> .</span></div>	</td>
	<td data-search="Absent in file"><div class="tooltip">Absent in file<span style="right: 14%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Exomiser_gene_pheno_score :</span> Absent in file<br><span class="commentTitle">Human_pheno_evidence :</span> Absent in file<br><span class="commentTitle">Mouse_pheno_evidence :</span> Absent in file<br><span class="commentTitle">Fish_pheno_evidence :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 28%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 42%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td ><div class="tooltip">.<span style="right: 57%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td ><div class="tooltip">.<span style="right: 71%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 85%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="Absent in file"><div class="tooltip">Absent in file<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> Absent in file</span></div>	</td>
</tr>
<tr class="fullsplit 2_142216368_148540744_DEL_1" >
	<td data-order="15" data-search="2_142216368_148540744_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr2:142216368-148540744&hgt.out1=submit&highlight=hg19.chr2:142216368-148540744#aaedff" target="_blank" rel="noopener noreferrer"">2_142216368_148540744_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :<br></span> 2_142216368_148540744_DEL_1<br><span class="commentTitle">SV_length :</span> -6324376</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 71%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ACMG_class :</span> .</span></div>	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 57%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 42%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFFFF" data-search="PABPC1P2"><div class="tooltip"><a href="https://www.genenames.org/tools/search/#!/all?query=PABPC1P2" target="_blank" rel="noopener noreferrer"">PABPC1P2</a><span style="left: 28%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> PABPC1P2<br><span class="commentTitle">LOEUF_bin :</span> .<br><span class="commentTitle">GnomAD_pLI :</span> .<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> .<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 14%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> UTR<br><span class="commentTitle">Tx :</span> NR_026904<br><span class="commentTitle">Exon_count :</span> 1<br><span class="commentTitle">Overlapped_tx_length :</span> 3934<br><span class="commentTitle">Overlapped_CDS_length :</span> 0<br><span class="commentTitle">Overlapped_CDS_percent :</span> 0<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_Nearest_SS :</span> .<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_Start :</span> 147344624<br><span class="commentTitle">Intersect_End :</span> 147348558</span></div>	</td>
	<td ><div class="tooltip">.<span style="left: 0%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidates :</span> .</span></div>	</td>
	<td data-search="Absent in file"><div class="tooltip">Absent in file<span style="right: 14%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Exomiser_gene_pheno_score :</span> Absent in file<br><span class="commentTitle">Human_pheno_evidence :</span> Absent in file<br><span class="commentTitle">Mouse_pheno_evidence :</span> Absent in file<br><span class="commentTitle">Fish_pheno_evidence :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 28%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 42%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td ><div class="tooltip">.<span style="right: 57%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td ><div class="tooltip">.<span style="right: 71%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 85%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="Absent in file"><div class="tooltip">Absent in file<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> Absent in file</span></div>	</td>
</tr>
<tr class="fullsplit 2_142216368_148540744_DEL_1" >
	<td data-order="16" data-search="2_142216368_148540744_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr2:142216368-148540744&hgt.out1=submit&highlight=hg19.chr2:142216368-148540744#aaedff" target="_blank" rel="noopener noreferrer"">2_142216368_148540744_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :<br></span> 2_142216368_148540744_DEL_1<br><span class="commentTitle">SV_length :</span> -6324376</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 71%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ACMG_class :</span> .</span></div>	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 57%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 42%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFFFF" data-search="LOC101928386"><div class="tooltip"><a href="https://www.genenames.org/tools/search/#!/all?query=LOC101928386" target="_blank" rel="noopener noreferrer"">LOC101928386</a><span style="left: 28%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> LOC101928386<br><span class="commentTitle">LOEUF_bin :</span> .<br><span class="commentTitle">GnomAD_pLI :</span> .<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> .<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 14%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> UTR<br><span class="commentTitle">Tx :</span> NR_110237<br><span class="commentTitle">Exon_count :</span> 3<br><span class="commentTitle">Overlapped_tx_length :</span> 27089<br><span class="commentTitle">Overlapped_CDS_length :</span> 0<br><span class="commentTitle">Overlapped_CDS_percent :</span> 0<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_Nearest_SS :</span> .<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_Start :</span> 144694633<br><span class="commentTitle">Intersect_End :</span> 144721722</span></div>	</td>
	<td ><div class="tooltip">.<span style="left: 0%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidates :</span> .</span></div>	</td>
	<td data-search="Absent in file"><div class="tooltip">Absent in file<span style="right: 14%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Exomiser_gene_pheno_score :</span> Absent in file<br><span class="commentTitle">Human_pheno_evidence :</span> Absent in file<br><span class="commentTitle">Mouse_pheno_evidence :</span> Absent in file<br><span class="commentTitle">Fish_pheno_evidence :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 28%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 42%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td ><div class="tooltip">.<span style="right: 57%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td ><div class="tooltip">.<span style="right: 71%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 85%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="Absent in file"><div class="tooltip">Absent in file<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> Absent in file</span></div>	</td>
</tr>
<tr class="fullsplit 2_142216368_148540744_DEL_1" >
	<td data-order="17" data-search="2_142216368_148540744_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr2:142216368-148540744&hgt.out1=submit&highlight=hg19.chr2:142216368-148540744#aaedff" target="_blank" rel="noopener noreferrer"">2_142216368_148540744_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :<br></span> 2_142216368_148540744_DEL_1<br><span class="commentTitle">SV_length :</span> -6324376</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 71%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ACMG_class :</span> .</span></div>	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 57%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 42%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFFFF" data-search="LINC01412"><div class="tooltip"><a href="https://www.genenames.org/tools/search/#!/all?query=LINC01412" target="_blank" rel="noopener noreferrer"">LINC01412</a><span style="left: 28%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> LINC01412<br><span class="commentTitle">LOEUF_bin :</span> .<br><span class="commentTitle">GnomAD_pLI :</span> .<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> .<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 14%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> UTR<br><span class="commentTitle">Tx :</span> NR_110570<br><span class="commentTitle">Exon_count :</span> 3<br><span class="commentTitle">Overlapped_tx_length :</span> 57567<br><span class="commentTitle">Overlapped_CDS_length :</span> 0<br><span class="commentTitle">Overlapped_CDS_percent :</span> 0<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_Nearest_SS :</span> .<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_Start :</span> 145279434<br><span class="commentTitle">Intersect_End :</span> 145337001</span></div>	</td>
	<td ><div class="tooltip">.<span style="left: 0%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidates :</span> .</span></div>	</td>
	<td data-search="Absent in file"><div class="tooltip">Absent in file<span style="right: 14%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Exomiser_gene_pheno_score :</span> Absent in file<br><span class="commentTitle">Human_pheno_evidence :</span> Absent in file<br><span class="commentTitle">Mouse_pheno_evidence :</span> Absent in file<br><span class="commentTitle">Fish_pheno_evidence :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 28%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 42%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td ><div class="tooltip">.<span style="right: 57%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td ><div class="tooltip">.<span style="right: 71%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 85%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="Absent in file"><div class="tooltip">Absent in file<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> Absent in file</span></div>	</td>
</tr>
<tr class="fullsplit 2_142216368_148540744_DEL_1" >
	<td data-order="18" data-search="2_142216368_148540744_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr2:142216368-148540744&hgt.out1=submit&highlight=hg19.chr2:142216368-148540744#aaedff" target="_blank" rel="noopener noreferrer"">2_142216368_148540744_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :<br></span> 2_142216368_148540744_DEL_1<br><span class="commentTitle">SV_length :</span> -6324376</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 71%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ACMG_class :</span> .</span></div>	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 57%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 42%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFFFF" data-search="LOC105373656"><div class="tooltip"><a href="https://www.genenames.org/tools/search/#!/all?query=LOC105373656" target="_blank" rel="noopener noreferrer"">LOC105373656</a><span style="left: 28%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> LOC105373656<br><span class="commentTitle">LOEUF_bin :</span> .<br><span class="commentTitle">GnomAD_pLI :</span> .<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> .<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 14%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> UTR<br><span class="commentTitle">Tx :</span> NR_136320<br><span class="commentTitle">Exon_count :</span> 2<br><span class="commentTitle">Overlapped_tx_length :</span> 895<br><span class="commentTitle">Overlapped_CDS_length :</span> 0<br><span class="commentTitle">Overlapped_CDS_percent :</span> 0<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_Nearest_SS :</span> .<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_Start :</span> 145281116<br><span class="commentTitle">Intersect_End :</span> 145282011</span></div>	</td>
	<td ><div class="tooltip">.<span style="left: 0%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidates :</span> .</span></div>	</td>
	<td data-search="Absent in file"><div class="tooltip">Absent in file<span style="right: 14%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Exomiser_gene_pheno_score :</span> Absent in file<br><span class="commentTitle">Human_pheno_evidence :</span> Absent in file<br><span class="commentTitle">Mouse_pheno_evidence :</span> Absent in file<br><span class="commentTitle">Fish_pheno_evidence :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 28%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 42%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td ><div class="tooltip">.<span style="right: 57%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td ><div class="tooltip">.<span style="right: 71%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 85%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="Absent in file"><div class="tooltip">Absent in file<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> Absent in file</span></div>	</td>
</tr>
<tr id="7_110276505_110478039_DEL_1" class="full" style="background-color:#F7D8DD" >
	<td data-order="19" data-search="7_110276505_110478039_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr7:110276505-110478039&hgt.out1=submit&highlight=hg19.chr7:110276505-110478039#aaedff" target="_blank" rel="noopener noreferrer"">7_110276505_110478039_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :<br></span> 7_110276505_110478039_DEL_1<br><span class="commentTitle">SV_length :</span> -201534<br><span class="commentTitle">Gene_count :</span> 1</span></div>	</td>
	<td data-search="3"><div class="tooltip">3<span style="left: 71%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ACMG_class :</span> 3<br><span class="commentTitle">AnnotSV_ranking_score :</span> 0.0<br><span class="commentTitle">AnnotSV_ranking_criteria :</span><br>1A: Contains protein-coding or other known functionally important elements. <br>3A (1 gene): 0â€“24 genes wholly or partially included<br></span></div>	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 57%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="full"><div class="tooltip">full<span style="left: 42%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> full</span></div>	</td>
	<td style="background-color:#FFFFFF" data-search="IMMP2L"><div class="tooltip">IMMP2L<span style="left: 28%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> IMMP2L<br><span class="commentTitle">LOEUF_bin :</span> .<br><span class="commentTitle">GnomAD_pLI :</span> 1.1165e-02<br><span class="commentTitle">ExAC_pLI :</span> 4.0761e-02<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 4.25<br><span class="commentTitle">ExAC_cnvZ :</span> -2.47162789907313<br><span class="commentTitle">ExAC_delZ :</span> -2.62354415217069<br><span class="commentTitle">ExAC_dupZ :</span> 0.194113758998799<br><span class="commentTitle">ExAC_synZ :</span> -0.54322591056105<br><span class="commentTitle">ExAC_misZ :</span> -0.0234666178325726<br><span class="commentTitle">DDD_status :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 14%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> .<br><span class="commentTitle">Overlapped_tx_length :</span> .<br><span class="commentTitle">Overlapped_CDS_length :</span> .</span></div>	</td>
	<td ><div class="tooltip">.<span style="left: 0%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidates :</span> .</span></div>	</td>
	<td data-search="Absent in file"><div class="tooltip">Absent in file<span style="right: 14%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Exomiser_gene_pheno_score :</span> Absent in file</span></div>	</td>
	<td data-search="LRRN3"><div class="tooltip">LRRN3<span style="right: 28%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> LRRN3</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 42%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td ><div class="tooltip">.<span style="right: 57%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td ><div class="tooltip">.<span style="right: 71%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 85%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .<br><span class="commentTitle">ENCODE_blacklist_left :</span> .<br><span class="commentTitle">SegDup_left :</span> .<br><span class="commentTitle">Repeat_coord_left :</span> 7:110276590-110276648<br><span class="commentTitle">Repeat_type_left :</span> L2b<br><span class="commentTitle">Gap_left :</span> .<br><span class="commentTitle">GC_content_left :</span> 0.350</span></div>	</td>
	<td data-search="Absent in file"><div class="tooltip">Absent in file<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> Absent in file<br><span class="commentTitle">ENCODE_blacklist_right :</span> .<br><span class="commentTitle">SegDup_right :</span> .<br><span class="commentTitle">Repeat_coord_right :</span> .<br><span class="commentTitle">Repeat_type_right :</span> .<br><span class="commentTitle">Gap_right :</span> .<br><span class="commentTitle">GC_content_right :</span> 0.360</span></div>	</td>
</tr>
<tr class="fullsplit 7_110276505_110478039_DEL_1" >
	<td data-order="20" data-search="7_110276505_110478039_DEL_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr7:110276505-110478039&hgt.out1=submit&highlight=hg19.chr7:110276505-110478039#aaedff" target="_blank" rel="noopener noreferrer"">7_110276505_110478039_DEL_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :<br></span> 7_110276505_110478039_DEL_1<br><span class="commentTitle">SV_length :</span> -201534</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 71%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ACMG_class :</span> .</span></div>	</td>
	<td data-search="DEL"><div class="tooltip">DEL<span style="left: 57%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DEL</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 42%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background: linear-gradient(-45deg,#BFFF00 50%, white 50% )" data-search="IMMP2L"><div class="tooltip"><a href="https://www.genenames.org/tools/search/#!/all?query=IMMP2L" target="_blank" rel="noopener noreferrer"">IMMP2L</a><span style="left: 28%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> IMMP2L<br><span class="commentTitle">LOEUF_bin :</span> 6<br><span class="commentTitle">GnomAD_pLI :</span> 1.1165e-02<br><span class="commentTitle">ExAC_pLI :</span> 4.0761e-02<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 4.25<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> -2.47162789907313<br><span class="commentTitle">ExAC_delZ :</span> -2.62354415217069<br><span class="commentTitle">ExAC_dupZ :</span> 0.194113758998799<br><span class="commentTitle">ExAC_synZ :</span> -0.54322591056105<br><span class="commentTitle">ExAC_misZ :</span> -0.0234666178325726<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="intron6-txEnd"><div class="tooltip">intron6-txEnd<span style="left: 14%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> intron6-txEnd<br><span class="commentTitle">Location2 :</span> CDS-3'UTR<br><span class="commentTitle">Tx :</span> NM_001244606<br><span class="commentTitle">Exon_count :</span> 7<br><span class="commentTitle">Overlapped_tx_length :</span> 175340<br><span class="commentTitle">Overlapped_CDS_length :</span> 120<br><span class="commentTitle">Overlapped_CDS_percent :</span> 22<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_Nearest_SS :</span> 48609<br><span class="commentTitle">Nearest_SS_type :</span> 5'<br><span class="commentTitle">Intersect_Start :</span> 110302699<br><span class="commentTitle">Intersect_End :</span> 110478039</span></div>	</td>
	<td ><div class="tooltip">.<span style="left: 0%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidates :</span> .</span></div>	</td>
	<td data-search="Absent in file"><div class="tooltip">Absent in file<span style="right: 14%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Exomiser_gene_pheno_score :</span> Absent in file<br><span class="commentTitle">Human_pheno_evidence :</span> Absent in file<br><span class="commentTitle">Mouse_pheno_evidence :</span> Absent in file<br><span class="commentTitle">Fish_pheno_evidence :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 28%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 42%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_loss_source :</span> .<br><span class="commentTitle">B_loss_coord :</span> .</span></div>	</td>
	<td ><div class="tooltip">.<span style="right: 57%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_loss_coord :</span> .<br><span class="commentTitle">P_loss_source :</span> .<br><span class="commentTitle">P_loss_phen :</span> .<br><span class="commentTitle">P_loss_hpo :</span> .</span></div>	</td>
	<td ><div class="tooltip">.<span style="right: 71%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 85%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="Absent in file"><div class="tooltip">Absent in file<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> Absent in file</span></div>	</td>
</tr>
<tr id="4_1563818_2158413_DUP_1" class="full" style="background-color:#DFE9FF" >
	<td data-order="21" data-search="4_1563818_2158413_DUP_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr4:1563818-2158413&hgt.out1=submit&highlight=hg19.chr4:1563818-2158413#aaedff" target="_blank" rel="noopener noreferrer"">4_1563818_2158413_DUP_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :<br></span> 4_1563818_2158413_DUP_1<br><span class="commentTitle">SV_length :</span> 594595<br><span class="commentTitle">Gene_count :</span> 13</span></div>	</td>
	<td data-search="3"><div class="tooltip">3<span style="left: 71%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ACMG_class :</span> 3<br><span class="commentTitle">AnnotSV_ranking_score :</span> 0.0<br><span class="commentTitle">AnnotSV_ranking_criteria :</span><br>1A: Contains protein-coding or other known functionally important elements. <br>3A (13 genes): 0â€“34 genes wholly or partially included<br></span></div>	</td>
	<td data-search="DUP"><div class="tooltip">DUP<span style="left: 57%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DUP</span></div>	</td>
	<td data-search="full"><div class="tooltip">full<span style="left: 42%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> full</span></div>	</td>
	<td style="background-color:#FFFFFF" data-search="C4orf48; FAM53A; FGFR3; LETM1; MIR943; NAT8L; NELFA; NSD2; POLN; SCARNA22; SLBP; TACC3; TMEM129"><div class="tooltip">C4orf48; FAM53A; FGFR3; LETM1; MIR943; NAT8L;[...]<span style="left: 28%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> C4orf48; FAM53A; FGFR3; LETM1; MIR943; NAT8L; NELFA; NSD2; POLN; SCARNA22; SLBP; TACC3; TMEM129<br><span class="commentTitle">LOEUF_bin :</span> .<br><span class="commentTitle">GnomAD_pLI :</span> .<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 14%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> .<br><span class="commentTitle">Overlapped_tx_length :</span> .<br><span class="commentTitle">Overlapped_CDS_length :</span> .</span></div>	</td>
	<td ><div class="tooltip">.<span style="left: 0%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidates :</span> .</span></div>	</td>
	<td data-search="Absent in file"><div class="tooltip">Absent in file<span style="right: 14%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Exomiser_gene_pheno_score :</span> Absent in file</span></div>	</td>
	<td data-search="CTBP1; CTBP1-DT; FAM193A; FGFRL1 (HI=0; TS=0); GAK; HAUS3; IDUA; MAEA; MFSD10; MXD4; NKX1-1; NOP14-AS1; PDE6B; RNF212; RNF4; SH3BP2; TMEM175; UVSSA; ZFYVE28"><div class="tooltip">CTBP1; CTBP1-DT; FAM193A; FGFRL1 (HI=0; TS=0)[...]<span style="right: 28%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> CTBP1; CTBP1-DT; FAM193A; FGFRL1 (HI=0; TS=0); GAK; HAUS3; IDUA; MAEA; MFSD10; MXD4; NKX1-1; NOP14-AS1; PDE6B; RNF212; RNF4; SH3BP2; TMEM175; UVSSA; ZFYVE28</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 42%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<br><span class="commentTitle">B_gain_coord :</span> .<br><span class="commentTitle">B_gain_source :</span> .<br><span class="commentTitle">B_gain_coord :</span> .</span></div>	</td>
	<td ><div class="tooltip">.<span style="right: 57%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<br><span class="commentTitle">P_gain_source :</span> .<br><span class="commentTitle">P_gain_phen :</span> .<br><span class="commentTitle">P_gain_hpo :</span> .<br><span class="commentTitle">P_gain_coord :</span> .<br><span class="commentTitle">P_gain_source :</span> .<br><span class="commentTitle">P_gain_phen :</span> .<br><span class="commentTitle">P_gain_hpo :</span> .</span></div>	</td>
	<td ><div class="tooltip">31<span style="right: 71%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> 31<br><span class="commentTitle">P_snvindel_phen :</span> 4p_partial_monosomy_syndrome; Absence_seizures; Achondroplasia; Bell-shaped_thorax; Birth_length_less_than_3rd_percentile; Bladder_carcinoma; Bowed_humerus; Bowing_of_the_long_bones; Camptodactyly-tall_stature-scoliosis-hearing_loss_syndrome; Carcinoma; Carcinoma_of_cervix; Carcinoma_of_colon; Cloverleaf_skull; Coronal_craniosynostosis; Craniosynostosis; Crouzon_syndrome; Crouzon_syndrome_with_acanthosis_nigricans; Disproportionate_short-limb_short_stature; Epidermal_nevus; FGFR3-related_disorder; Facial_asymmetry; Femoral_bowing; Growth_delay; Hypochondroplasia; Inborn_genetic_diseases; Infantile_axial_hypotonia; Lethal_short-limbed_short_stature; Lethal_skeletal_dysplasia; Levy-Hollister_syndrome; Lower_limb_undergrowth; Lung_adenocarcinoma; Macrocephalus; Malignant_tumor_of_testis; Muenke_syndrome; Multiple_myeloma; Narrow_chest; Papillary_renal_cell_carcinoma,_sporadic; Platyspondyly; Polyhydramnios; Saethre-Chotzen_syndrome; Seborrheic_keratosis; Seizures; Severe_achondroplasia_with_developmental_delay_and_acanthosis_nigricans; Short_long_bone; Short_ribs; Short_stature; Skeletal_dysplasia; Skeletal_dysplasia_with_acanthosis_nigricans; Small_for_gestational_age; Spermatocytic_seminoma; Squamous_cell_carcinoma_of_the_head_and_neck; Squamous_cell_lung_carcinoma; Thanatophoric_dysplasia,_type_2; Thanatophoric_dysplasia_type_1; Transitional_cell_carcinoma_of_the_bladder; Unilateral_renal_agenesis; Upper_limb_undergrowth; Urinary_bladder_cancer; Wolf-Hirschhorn_like_syndrome; not_specified</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 85%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .<br><span class="commentTitle">ENCODE_blacklist_left :</span> .<br><span class="commentTitle">SegDup_left :</span> .<br><span class="commentTitle">Repeat_coord_left :</span> .<br><span class="commentTitle">Repeat_type_left :</span> .<br><span class="commentTitle">Gap_left :</span> .<br><span class="commentTitle">GC_content_left :</span> 0.535</span></div>	</td>
	<td data-search="Absent in file"><div class="tooltip">Absent in file<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> Absent in file<br><span class="commentTitle">ENCODE_blacklist_right :</span> .<br><span class="commentTitle">SegDup_right :</span> .<br><span class="commentTitle">Repeat_coord_right :</span> .<br><span class="commentTitle">Repeat_type_right :</span> .<br><span class="commentTitle">Gap_right :</span> .<br><span class="commentTitle">GC_content_right :</span> 0.380</span></div>	</td>
</tr>
<tr class="fullsplit 4_1563818_2158413_DUP_1" >
	<td data-order="22" data-search="4_1563818_2158413_DUP_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr4:1563818-2158413&hgt.out1=submit&highlight=hg19.chr4:1563818-2158413#aaedff" target="_blank" rel="noopener noreferrer"">4_1563818_2158413_DUP_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :<br></span> 4_1563818_2158413_DUP_1<br><span class="commentTitle">SV_length :</span> 594595</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 71%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ACMG_class :</span> .</span></div>	</td>
	<td data-search="DUP"><div class="tooltip">DUP<span style="left: 57%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DUP</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 42%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FF6600" data-search="NELFA"><div class="tooltip"><a href="https://www.genenames.org/tools/search/#!/all?query=NELFA" target="_blank" rel="noopener noreferrer"">NELFA</a><span style="left: 28%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> NELFA<br><span class="commentTitle">LOEUF_bin :</span> 2<br><span class="commentTitle">GnomAD_pLI :</span> 1.3469e-01<br><span class="commentTitle">ExAC_pLI :</span> 1.6689e-01<br><span class="commentTitle">HI :</span> 0<br><span class="commentTitle">TS :</span> 0<br><span class="commentTitle">DDD_HI_percent :</span> 62.86<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> 0.486612084123194<br><span class="commentTitle">ExAC_delZ :</span> 0.0789021097033232<br><span class="commentTitle">ExAC_dupZ :</span> 0.586448065563417<br><span class="commentTitle">ExAC_synZ :</span> -0.704500724266535<br><span class="commentTitle">ExAC_misZ :</span> 2.35326609523799<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 14%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_005663<br><span class="commentTitle">Exon_count :</span> 11<br><span class="commentTitle">Overlapped_tx_length :</span> 26251<br><span class="commentTitle">Overlapped_CDS_length :</span> 1587<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_Nearest_SS :</span> .<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_Start :</span> 1984450<br><span class="commentTitle">Intersect_End :</span> 2010701</span></div>	</td>
	<td ><div class="tooltip">.<span style="left: 0%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidates :</span> .</span></div>	</td>
	<td data-search="Absent in file"><div class="tooltip">Absent in file<span style="right: 14%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Exomiser_gene_pheno_score :</span> Absent in file<br><span class="commentTitle">Human_pheno_evidence :</span> Absent in file<br><span class="commentTitle">Mouse_pheno_evidence :</span> Absent in file<br><span class="commentTitle">Fish_pheno_evidence :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 28%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 42%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<br><span class="commentTitle">B_gain_coord :</span> .<br><span class="commentTitle">B_gain_source :</span> .<br><span class="commentTitle">B_gain_coord :</span> .</span></div>	</td>
	<td ><div class="tooltip">.<span style="right: 57%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<br><span class="commentTitle">P_gain_source :</span> .<br><span class="commentTitle">P_gain_phen :</span> .<br><span class="commentTitle">P_gain_hpo :</span> .<br><span class="commentTitle">P_gain_coord :</span> .<br><span class="commentTitle">P_gain_source :</span> .<br><span class="commentTitle">P_gain_phen :</span> .<br><span class="commentTitle">P_gain_hpo :</span> .</span></div>	</td>
	<td ><div class="tooltip">.<span style="right: 71%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 85%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="Absent in file"><div class="tooltip">Absent in file<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> Absent in file</span></div>	</td>
</tr>
<tr class="fullsplit 4_1563818_2158413_DUP_1" >
	<td data-order="23" data-search="4_1563818_2158413_DUP_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr4:1563818-2158413&hgt.out1=submit&highlight=hg19.chr4:1563818-2158413#aaedff" target="_blank" rel="noopener noreferrer"">4_1563818_2158413_DUP_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :<br></span> 4_1563818_2158413_DUP_1<br><span class="commentTitle">SV_length :</span> 594595</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 71%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ACMG_class :</span> .</span></div>	</td>
	<td data-search="DUP"><div class="tooltip">DUP<span style="left: 57%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DUP</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 42%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FF6600" data-search="NAT8L"><div class="tooltip"><a href="https://www.genenames.org/tools/search/#!/all?query=NAT8L" target="_blank" rel="noopener noreferrer"">NAT8L</a><span style="left: 28%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> NAT8L<br><span class="commentTitle">LOEUF_bin :</span> 2<br><span class="commentTitle">GnomAD_pLI :</span> 8.6458e-01<br><span class="commentTitle">ExAC_pLI :</span> 3.2468e-01<br><span class="commentTitle">HI :</span> 30<br><span class="commentTitle">TS :</span> 0<br><span class="commentTitle">DDD_HI_percent :</span> 40.87<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> 0.555421231184509<br><span class="commentTitle">ExAC_delZ :</span> 0.098456637925367<br><span class="commentTitle">ExAC_dupZ :</span> 0.547035422647778<br><span class="commentTitle">ExAC_synZ :</span> -0.17378562567541<br><span class="commentTitle">ExAC_misZ :</span> 2.72200675038499<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 14%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_178557<br><span class="commentTitle">Exon_count :</span> 3<br><span class="commentTitle">Overlapped_tx_length :</span> 9763<br><span class="commentTitle">Overlapped_CDS_length :</span> 909<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_Nearest_SS :</span> .<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_Start :</span> 2061053<br><span class="commentTitle">Intersect_End :</span> 2070816</span></div>	</td>
	<td ><div class="tooltip">?N-acetylaspartate deficiency, 614063 (3)<span style="left: 0%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> ?N-acetylaspartate deficiency, 614063 (3)<br><span class="commentTitle">OMIM_ID :</span> <a href="https://www.omim.org/entry/610647" target="_blank" rel="noopener noreferrer" style="color:white">610647</a><br><span class="commentTitle">OMIM_inheritance :</span> AR<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidates :</span> yes</span></div>	</td>
	<td data-search="Absent in file"><div class="tooltip">Absent in file<span style="right: 14%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Exomiser_gene_pheno_score :</span> Absent in file<br><span class="commentTitle">Human_pheno_evidence :</span> Absent in file<br><span class="commentTitle">Mouse_pheno_evidence :</span> Absent in file<br><span class="commentTitle">Fish_pheno_evidence :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 28%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="gnomAD-SV_v2.1_DUP_4_10952"><div class="tooltip">gnomAD-SV_v2.1_DUP_4_10952<span style="right: 42%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> gnomAD-SV_v2.1_DUP_4_10952<br><span class="commentTitle">B_gain_coord :</span> 4:2060724-2076000<br><span class="commentTitle">B_gain_source :</span> gnomAD-SV_v2.1_DUP_4_10952<br><span class="commentTitle">B_gain_coord :</span> 4:2060724-2076000</span></div>	</td>
	<td ><div class="tooltip">.<span style="right: 57%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<br><span class="commentTitle">P_gain_source :</span> .<br><span class="commentTitle">P_gain_phen :</span> .<br><span class="commentTitle">P_gain_hpo :</span> .<br><span class="commentTitle">P_gain_coord :</span> .<br><span class="commentTitle">P_gain_source :</span> .<br><span class="commentTitle">P_gain_phen :</span> .<br><span class="commentTitle">P_gain_hpo :</span> .</span></div>	</td>
	<td ><div class="tooltip">.<span style="right: 71%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 85%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="Absent in file"><div class="tooltip">Absent in file<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> Absent in file</span></div>	</td>
</tr>
<tr class="fullsplit 4_1563818_2158413_DUP_1" >
	<td data-order="24" data-search="4_1563818_2158413_DUP_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr4:1563818-2158413&hgt.out1=submit&highlight=hg19.chr4:1563818-2158413#aaedff" target="_blank" rel="noopener noreferrer"">4_1563818_2158413_DUP_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :<br></span> 4_1563818_2158413_DUP_1<br><span class="commentTitle">SV_length :</span> 594595</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 71%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ACMG_class :</span> .</span></div>	</td>
	<td data-search="DUP"><div class="tooltip">DUP<span style="left: 57%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DUP</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 42%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FF9900" data-search="TACC3"><div class="tooltip"><a href="https://www.genenames.org/tools/search/#!/all?query=TACC3" target="_blank" rel="noopener noreferrer"">TACC3</a><span style="left: 28%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> TACC3<br><span class="commentTitle">LOEUF_bin :</span> 3<br><span class="commentTitle">GnomAD_pLI :</span> 2.7082e-07<br><span class="commentTitle">ExAC_pLI :</span> 9.1952e-02<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 92.5<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> -0.98783763752124<br><span class="commentTitle">ExAC_delZ :</span> -0.738565549612952<br><span class="commentTitle">ExAC_dupZ :</span> -1.04002887445022<br><span class="commentTitle">ExAC_synZ :</span> -1.00228696058413<br><span class="commentTitle">ExAC_misZ :</span> -1.35276544118498<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 14%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_006342<br><span class="commentTitle">Exon_count :</span> 16<br><span class="commentTitle">Overlapped_tx_length :</span> 23650<br><span class="commentTitle">Overlapped_CDS_length :</span> 2517<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_Nearest_SS :</span> .<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_Start :</span> 1723247<br><span class="commentTitle">Intersect_End :</span> 1746897</span></div>	</td>
	<td ><div class="tooltip">.<span style="left: 0%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidates :</span> .</span></div>	</td>
	<td data-search="Absent in file"><div class="tooltip">Absent in file<span style="right: 14%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Exomiser_gene_pheno_score :</span> Absent in file<br><span class="commentTitle">Human_pheno_evidence :</span> Absent in file<br><span class="commentTitle">Mouse_pheno_evidence :</span> Absent in file<br><span class="commentTitle">Fish_pheno_evidence :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 28%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 42%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<br><span class="commentTitle">B_gain_coord :</span> .<br><span class="commentTitle">B_gain_source :</span> .<br><span class="commentTitle">B_gain_coord :</span> .</span></div>	</td>
	<td ><div class="tooltip">.<span style="right: 57%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<br><span class="commentTitle">P_gain_source :</span> .<br><span class="commentTitle">P_gain_phen :</span> .<br><span class="commentTitle">P_gain_hpo :</span> .<br><span class="commentTitle">P_gain_coord :</span> .<br><span class="commentTitle">P_gain_source :</span> .<br><span class="commentTitle">P_gain_phen :</span> .<br><span class="commentTitle">P_gain_hpo :</span> .</span></div>	</td>
	<td ><div class="tooltip">.<span style="right: 71%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 85%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="Absent in file"><div class="tooltip">Absent in file<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> Absent in file</span></div>	</td>
</tr>
<tr class="fullsplit 4_1563818_2158413_DUP_1" >
	<td data-order="25" data-search="4_1563818_2158413_DUP_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr4:1563818-2158413&hgt.out1=submit&highlight=hg19.chr4:1563818-2158413#aaedff" target="_blank" rel="noopener noreferrer"">4_1563818_2158413_DUP_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :<br></span> 4_1563818_2158413_DUP_1<br><span class="commentTitle">SV_length :</span> 594595</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 71%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ACMG_class :</span> .</span></div>	</td>
	<td data-search="DUP"><div class="tooltip">DUP<span style="left: 57%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DUP</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 42%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FF9900" data-search="LETM1"><div class="tooltip"><a href="https://www.genenames.org/tools/search/#!/all?query=LETM1" target="_blank" rel="noopener noreferrer"">LETM1</a><span style="left: 28%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> LETM1<br><span class="commentTitle">LOEUF_bin :</span> 3<br><span class="commentTitle">GnomAD_pLI :</span> 1.2141e-06<br><span class="commentTitle">ExAC_pLI :</span> 8.1055e-02<br><span class="commentTitle">HI :</span> 0<br><span class="commentTitle">TS :</span> 0<br><span class="commentTitle">DDD_HI_percent :</span> 63.78<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> -0.0279109913872425<br><span class="commentTitle">ExAC_delZ :</span> 0.214548231055587<br><span class="commentTitle">ExAC_dupZ :</span> -0.175454101774909<br><span class="commentTitle">ExAC_synZ :</span> -1.46983950545222<br><span class="commentTitle">ExAC_misZ :</span> 1.90036967420169<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 14%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_012318<br><span class="commentTitle">Exon_count :</span> 14<br><span class="commentTitle">Overlapped_tx_length :</span> 44678<br><span class="commentTitle">Overlapped_CDS_length :</span> 2220<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_Nearest_SS :</span> .<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_Start :</span> 1813205<br><span class="commentTitle">Intersect_End :</span> 1857883</span></div>	</td>
	<td ><div class="tooltip">.<span style="left: 0%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidates :</span> .</span></div>	</td>
	<td data-search="Absent in file"><div class="tooltip">Absent in file<span style="right: 14%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Exomiser_gene_pheno_score :</span> Absent in file<br><span class="commentTitle">Human_pheno_evidence :</span> Absent in file<br><span class="commentTitle">Mouse_pheno_evidence :</span> Absent in file<br><span class="commentTitle">Fish_pheno_evidence :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 28%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 42%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<br><span class="commentTitle">B_gain_coord :</span> .<br><span class="commentTitle">B_gain_source :</span> .<br><span class="commentTitle">B_gain_coord :</span> .</span></div>	</td>
	<td ><div class="tooltip">.<span style="right: 57%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<br><span class="commentTitle">P_gain_source :</span> .<br><span class="commentTitle">P_gain_phen :</span> .<br><span class="commentTitle">P_gain_hpo :</span> .<br><span class="commentTitle">P_gain_coord :</span> .<br><span class="commentTitle">P_gain_source :</span> .<br><span class="commentTitle">P_gain_phen :</span> .<br><span class="commentTitle">P_gain_hpo :</span> .</span></div>	</td>
	<td ><div class="tooltip">.<span style="right: 71%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 85%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="Absent in file"><div class="tooltip">Absent in file<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> Absent in file</span></div>	</td>
</tr>
<tr class="fullsplit 4_1563818_2158413_DUP_1" >
	<td data-order="26" data-search="4_1563818_2158413_DUP_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr4:1563818-2158413&hgt.out1=submit&highlight=hg19.chr4:1563818-2158413#aaedff" target="_blank" rel="noopener noreferrer"">4_1563818_2158413_DUP_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :<br></span> 4_1563818_2158413_DUP_1<br><span class="commentTitle">SV_length :</span> 594595</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 71%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ACMG_class :</span> .</span></div>	</td>
	<td data-search="DUP"><div class="tooltip">DUP<span style="left: 57%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DUP</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 42%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FF9900" data-search="FGFR3"><div class="tooltip"><a href="https://www.genenames.org/tools/search/#!/all?query=FGFR3" target="_blank" rel="noopener noreferrer"">FGFR3</a><span style="left: 28%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> FGFR3<br><span class="commentTitle">LOEUF_bin :</span> 3<br><span class="commentTitle">GnomAD_pLI :</span> 1.6432e-05<br><span class="commentTitle">ExAC_pLI :</span> 2.9091e-03<br><span class="commentTitle">HI :</span> 0<br><span class="commentTitle">TS :</span> 0<br><span class="commentTitle">DDD_HI_percent :</span> 6.4<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> -0.538573860699497<br><span class="commentTitle">ExAC_delZ :</span> -0.683349552436101<br><span class="commentTitle">ExAC_dupZ :</span> -0.392116289350703<br><span class="commentTitle">ExAC_synZ :</span> -2.22163439002498<br><span class="commentTitle">ExAC_misZ :</span> 1.36559781959522<br><span class="commentTitle">DDD_status :</span> confirmed/confirmed/confirmed/confirmed/confirmed/confirmed/confirmed/confirmed<br><span class="commentTitle">DDD_mode :</span> monoallelic/monoallelic/monoallelic/monoallelic/monoallelic/monoallelic/monoallelic/monoallelic<br><span class="commentTitle">DDD_consequence :</span> all missense/in frame/activating/activating/activating/activating/all missense/in frame/activating/activating<br><span class="commentTitle">DDD_disease :</span> LACRIMO-AURICULO-DENTO-DIGITAL SYNDROME/MUENKE SYNDROME/CROUZON SYNDROME WITH ACANTHOSIS NIGRICANS/ACHONDROPLASIA/THANATOPHORIC DYSPLASIA TYPE 2/CAMPTODACTYLY TALL STATURE AND HEARING LOSS SYNDROME/HYPOCHONDROPLASIA/THANATOPHORIC DYSPLASIA TYPE 1<br><span class="commentTitle">DDD_pmid :</span> 16501574; 28483234//7493034/16411219; 8078586; 7758520; 16912704; 7913883/7773297/27139183; 17033969/11055896; 7670477; 16912704; 10215410; 9452043; 8589686/19449430; 7647778; 7773297; 8845844</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 14%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_001163213<br><span class="commentTitle">Exon_count :</span> 18<br><span class="commentTitle">Overlapped_tx_length :</span> 15561<br><span class="commentTitle">Overlapped_CDS_length :</span> 2427<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_Nearest_SS :</span> .<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_Start :</span> 1795038<br><span class="commentTitle">Intersect_End :</span> 1810599</span></div>	</td>
	<td ><div class="tooltip">Muenke syndrome, 602849 (3)/ Nevus, epidermal[...]<span style="left: 0%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> Muenke syndrome, 602849 (3)/ Nevus, epidermal, somatic, 162900 (3)/ Thanatophoric dysplasia, type II, 187601 (3)/ Bladder cancer, somatic, 109800 (3)/ CATSHL syndrome, 610474 (3)/ Crouzon syndrome with acanthosis nigricans, 612247 (3)/ Hypochondroplasia, 146000 (3)/ LADD syndrome, 149730 (3)/ Achondroplasia, 100800 (3)/ Thanatophoric dysplasia, type I, 187600 (3)/ Colorectal cancer, somatic, 114500 (3)/ Spermatocytic seminoma, somatic, 273300 (3)/ Cervical cancer, somatic, 603956 (3)/ SADDAN, 616482 (3)<br><span class="commentTitle">OMIM_ID :</span> <a href="https://www.omim.org/entry/134934" target="_blank" rel="noopener noreferrer" style="color:white">134934</a><br><span class="commentTitle">OMIM_inheritance :</span> AD//AD//AR, AD/AD/AD/AD/AD/AD////AD<br><span class="commentTitle">OMIM_morbid :</span> yes<br><span class="commentTitle">OMIM_morbid_candidates :</span> .</span></div>	</td>
	<td data-search="Absent in file"><div class="tooltip">Absent in file<span style="right: 14%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Exomiser_gene_pheno_score :</span> Absent in file<br><span class="commentTitle">Human_pheno_evidence :</span> Absent in file<br><span class="commentTitle">Mouse_pheno_evidence :</span> Absent in file<br><span class="commentTitle">Fish_pheno_evidence :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 28%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 42%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<br><span class="commentTitle">B_gain_coord :</span> .<br><span class="commentTitle">B_gain_source :</span> .<br><span class="commentTitle">B_gain_coord :</span> .</span></div>	</td>
	<td ><div class="tooltip">.<span style="right: 57%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<br><span class="commentTitle">P_gain_source :</span> .<br><span class="commentTitle">P_gain_phen :</span> .<br><span class="commentTitle">P_gain_hpo :</span> .<br><span class="commentTitle">P_gain_coord :</span> .<br><span class="commentTitle">P_gain_source :</span> .<br><span class="commentTitle">P_gain_phen :</span> .<br><span class="commentTitle">P_gain_hpo :</span> .</span></div>	</td>
	<td ><div class="tooltip">22<span style="right: 71%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> 22<br><span class="commentTitle">P_snvindel_phen :</span> Absence_seizures; Achondroplasia; Bell-shaped_thorax; Birth_length_less_than_3rd_percentile; Bladder_carcinoma; Bowed_humerus; Bowing_of_the_long_bones; Camptodactyly-tall_stature-scoliosis-hearing_loss_syndrome; Carcinoma; Carcinoma_of_cervix; Carcinoma_of_colon; Cloverleaf_skull; Coronal_craniosynostosis; Craniosynostosis; Crouzon_syndrome; Crouzon_syndrome_with_acanthosis_nigricans; Disproportionate_short-limb_short_stature; Epidermal_nevus; FGFR3-related_disorder; Facial_asymmetry; Femoral_bowing; Growth_delay; Hypochondroplasia; Inborn_genetic_diseases; Infantile_axial_hypotonia; Lethal_short-limbed_short_stature; Lethal_skeletal_dysplasia; Levy-Hollister_syndrome; Lower_limb_undergrowth; Lung_adenocarcinoma; Macrocephalus; Malignant_tumor_of_testis; Muenke_syndrome; Multiple_myeloma; Narrow_chest; Papillary_renal_cell_carcinoma,_sporadic; Platyspondyly; Polyhydramnios; Saethre-Chotzen_syndrome; Seborrheic_keratosis; Seizures; Severe_achondroplasia_with_developmental_delay_and_acanthosis_nigricans; Short_long_bone; Short_ribs; Short_stature; Skeletal_dysplasia; Skeletal_dysplasia_with_acanthosis_nigricans; Small_for_gestational_age; Spermatocytic_seminoma; Squamous_cell_carcinoma_of_the_head_and_neck; Squamous_cell_lung_carcinoma; Thanatophoric_dysplasia,_type_2; Thanatophoric_dysplasia_type_1; Transitional_cell_carcinoma_of_the_bladder; Unilateral_renal_agenesis; Upper_limb_undergrowth; Urinary_bladder_cancer; not_specified</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 85%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="Absent in file"><div class="tooltip">Absent in file<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> Absent in file</span></div>	</td>
</tr>
<tr class="fullsplit 4_1563818_2158413_DUP_1" >
	<td data-order="27" data-search="4_1563818_2158413_DUP_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr4:1563818-2158413&hgt.out1=submit&highlight=hg19.chr4:1563818-2158413#aaedff" target="_blank" rel="noopener noreferrer"">4_1563818_2158413_DUP_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :<br></span> 4_1563818_2158413_DUP_1<br><span class="commentTitle">SV_length :</span> 594595</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 71%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ACMG_class :</span> .</span></div>	</td>
	<td data-search="DUP"><div class="tooltip">DUP<span style="left: 57%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DUP</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 42%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFCC00" data-search="SLBP"><div class="tooltip"><a href="https://www.genenames.org/tools/search/#!/all?query=SLBP" target="_blank" rel="noopener noreferrer"">SLBP</a><span style="left: 28%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> SLBP<br><span class="commentTitle">LOEUF_bin :</span> 4<br><span class="commentTitle">GnomAD_pLI :</span> 6.8148e-02<br><span class="commentTitle">ExAC_pLI :</span> 5.1566e-01<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 41.08<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> -0.660904930034766<br><span class="commentTitle">ExAC_delZ :</span> -0.996144409589105<br><span class="commentTitle">ExAC_dupZ :</span> -0.44060461568473<br><span class="commentTitle">ExAC_synZ :</span> -0.443208130978936<br><span class="commentTitle">ExAC_misZ :</span> 0.50373477013136<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 14%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_006527<br><span class="commentTitle">Exon_count :</span> 8<br><span class="commentTitle">Overlapped_tx_length :</span> 19589<br><span class="commentTitle">Overlapped_CDS_length :</span> 813<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_Nearest_SS :</span> .<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_Start :</span> 1694457<br><span class="commentTitle">Intersect_End :</span> 1714046</span></div>	</td>
	<td ><div class="tooltip">.<span style="left: 0%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidates :</span> .</span></div>	</td>
	<td data-search="Absent in file"><div class="tooltip">Absent in file<span style="right: 14%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Exomiser_gene_pheno_score :</span> Absent in file<br><span class="commentTitle">Human_pheno_evidence :</span> Absent in file<br><span class="commentTitle">Mouse_pheno_evidence :</span> Absent in file<br><span class="commentTitle">Fish_pheno_evidence :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 28%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 42%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<br><span class="commentTitle">B_gain_coord :</span> .<br><span class="commentTitle">B_gain_source :</span> .<br><span class="commentTitle">B_gain_coord :</span> .</span></div>	</td>
	<td ><div class="tooltip">.<span style="right: 57%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<br><span class="commentTitle">P_gain_source :</span> .<br><span class="commentTitle">P_gain_phen :</span> .<br><span class="commentTitle">P_gain_hpo :</span> .<br><span class="commentTitle">P_gain_coord :</span> .<br><span class="commentTitle">P_gain_source :</span> .<br><span class="commentTitle">P_gain_phen :</span> .<br><span class="commentTitle">P_gain_hpo :</span> .</span></div>	</td>
	<td ><div class="tooltip">.<span style="right: 71%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 85%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="Absent in file"><div class="tooltip">Absent in file<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> Absent in file</span></div>	</td>
</tr>
<tr class="fullsplit 4_1563818_2158413_DUP_1" >
	<td data-order="28" data-search="4_1563818_2158413_DUP_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr4:1563818-2158413&hgt.out1=submit&highlight=hg19.chr4:1563818-2158413#aaedff" target="_blank" rel="noopener noreferrer"">4_1563818_2158413_DUP_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :<br></span> 4_1563818_2158413_DUP_1<br><span class="commentTitle">SV_length :</span> 594595</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 71%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ACMG_class :</span> .</span></div>	</td>
	<td data-search="DUP"><div class="tooltip">DUP<span style="left: 57%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DUP</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 42%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#BFFF00" data-search="FAM53A"><div class="tooltip"><a href="https://www.genenames.org/tools/search/#!/all?query=FAM53A" target="_blank" rel="noopener noreferrer"">FAM53A</a><span style="left: 28%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> FAM53A<br><span class="commentTitle">LOEUF_bin :</span> 6<br><span class="commentTitle">GnomAD_pLI :</span> 3.5705e-03<br><span class="commentTitle">ExAC_pLI :</span> 3.0070e-05<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 86.49<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> -0.576730190868019<br><span class="commentTitle">ExAC_misZ :</span> -0.56780381579822<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 14%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_001174070<br><span class="commentTitle">Exon_count :</span> 5<br><span class="commentTitle">Overlapped_tx_length :</span> 44433<br><span class="commentTitle">Overlapped_CDS_length :</span> 1197<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_Nearest_SS :</span> .<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_Start :</span> 1641607<br><span class="commentTitle">Intersect_End :</span> 1686040</span></div>	</td>
	<td ><div class="tooltip">.<span style="left: 0%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidates :</span> .</span></div>	</td>
	<td data-search="Absent in file"><div class="tooltip">Absent in file<span style="right: 14%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Exomiser_gene_pheno_score :</span> Absent in file<br><span class="commentTitle">Human_pheno_evidence :</span> Absent in file<br><span class="commentTitle">Mouse_pheno_evidence :</span> Absent in file<br><span class="commentTitle">Fish_pheno_evidence :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 28%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 42%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<br><span class="commentTitle">B_gain_coord :</span> .<br><span class="commentTitle">B_gain_source :</span> .<br><span class="commentTitle">B_gain_coord :</span> .</span></div>	</td>
	<td ><div class="tooltip">.<span style="right: 57%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<br><span class="commentTitle">P_gain_source :</span> .<br><span class="commentTitle">P_gain_phen :</span> .<br><span class="commentTitle">P_gain_hpo :</span> .<br><span class="commentTitle">P_gain_coord :</span> .<br><span class="commentTitle">P_gain_source :</span> .<br><span class="commentTitle">P_gain_phen :</span> .<br><span class="commentTitle">P_gain_hpo :</span> .</span></div>	</td>
	<td ><div class="tooltip">.<span style="right: 71%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 85%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="Absent in file"><div class="tooltip">Absent in file<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> Absent in file</span></div>	</td>
</tr>
<tr class="fullsplit 4_1563818_2158413_DUP_1" >
	<td data-order="29" data-search="4_1563818_2158413_DUP_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr4:1563818-2158413&hgt.out1=submit&highlight=hg19.chr4:1563818-2158413#aaedff" target="_blank" rel="noopener noreferrer"">4_1563818_2158413_DUP_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :<br></span> 4_1563818_2158413_DUP_1<br><span class="commentTitle">SV_length :</span> 594595</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 71%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ACMG_class :</span> .</span></div>	</td>
	<td data-search="DUP"><div class="tooltip">DUP<span style="left: 57%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DUP</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 42%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background: linear-gradient(-45deg,#7FFF00 50%, white 50% )" data-search="POLN"><div class="tooltip"><a href="https://www.genenames.org/tools/search/#!/all?query=POLN" target="_blank" rel="noopener noreferrer"">POLN</a><span style="left: 28%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> POLN<br><span class="commentTitle">LOEUF_bin :</span> 7<br><span class="commentTitle">GnomAD_pLI :</span> 9.4880e-41<br><span class="commentTitle">ExAC_pLI :</span> 2.9966e-32<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 74.11<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> 0.298427608261686<br><span class="commentTitle">ExAC_delZ :</span> 1.28008098953073<br><span class="commentTitle">ExAC_dupZ :</span> -0.213680271829885<br><span class="commentTitle">ExAC_synZ :</span> 0.0230368692484615<br><span class="commentTitle">ExAC_misZ :</span> -1.08931776314455<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="intron16-txEnd"><div class="tooltip">intron16-txEnd<span style="left: 14%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> intron16-txEnd<br><span class="commentTitle">Location2 :</span> CDS-3'UTR<br><span class="commentTitle">Tx :</span> NM_181808<br><span class="commentTitle">Exon_count :</span> 26<br><span class="commentTitle">Overlapped_tx_length :</span> 84769<br><span class="commentTitle">Overlapped_CDS_length :</span> 972<br><span class="commentTitle">Overlapped_CDS_percent :</span> 35<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_Nearest_SS :</span> 101<br><span class="commentTitle">Nearest_SS_type :</span> 5'<br><span class="commentTitle">Intersect_Start :</span> 2073644<br><span class="commentTitle">Intersect_End :</span> 2158413</span></div>	</td>
	<td ><div class="tooltip">.<span style="left: 0%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidates :</span> .</span></div>	</td>
	<td data-search="Absent in file"><div class="tooltip">Absent in file<span style="right: 14%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Exomiser_gene_pheno_score :</span> Absent in file<br><span class="commentTitle">Human_pheno_evidence :</span> Absent in file<br><span class="commentTitle">Mouse_pheno_evidence :</span> Absent in file<br><span class="commentTitle">Fish_pheno_evidence :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 28%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 42%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<br><span class="commentTitle">B_gain_coord :</span> .<br><span class="commentTitle">B_gain_source :</span> .<br><span class="commentTitle">B_gain_coord :</span> .</span></div>	</td>
	<td ><div class="tooltip">.<span style="right: 57%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<br><span class="commentTitle">P_gain_source :</span> .<br><span class="commentTitle">P_gain_phen :</span> .<br><span class="commentTitle">P_gain_hpo :</span> .<br><span class="commentTitle">P_gain_coord :</span> .<br><span class="commentTitle">P_gain_source :</span> .<br><span class="commentTitle">P_gain_phen :</span> .<br><span class="commentTitle">P_gain_hpo :</span> .</span></div>	</td>
	<td ><div class="tooltip">.<span style="right: 71%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 85%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="Absent in file"><div class="tooltip">Absent in file<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> Absent in file</span></div>	</td>
</tr>
<tr class="fullsplit 4_1563818_2158413_DUP_1" >
	<td data-order="30" data-search="4_1563818_2158413_DUP_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr4:1563818-2158413&hgt.out1=submit&highlight=hg19.chr4:1563818-2158413#aaedff" target="_blank" rel="noopener noreferrer"">4_1563818_2158413_DUP_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :<br></span> 4_1563818_2158413_DUP_1<br><span class="commentTitle">SV_length :</span> 594595</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 71%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ACMG_class :</span> .</span></div>	</td>
	<td data-search="DUP"><div class="tooltip">DUP<span style="left: 57%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DUP</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 42%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#7FFF00" data-search="C4orf48"><div class="tooltip"><a href="https://www.genenames.org/tools/search/#!/all?query=C4orf48" target="_blank" rel="noopener noreferrer"">C4orf48</a><span style="left: 28%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> C4orf48<br><span class="commentTitle">LOEUF_bin :</span> 7<br><span class="commentTitle">GnomAD_pLI :</span> 5.1423e-01<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 80.21<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 14%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_001168243<br><span class="commentTitle">Exon_count :</span> 4<br><span class="commentTitle">Overlapped_tx_length :</span> 1978<br><span class="commentTitle">Overlapped_CDS_length :</span> 387<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_Nearest_SS :</span> .<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_Start :</span> 2043719<br><span class="commentTitle">Intersect_End :</span> 2045697</span></div>	</td>
	<td ><div class="tooltip">.<span style="left: 0%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidates :</span> .</span></div>	</td>
	<td data-search="Absent in file"><div class="tooltip">Absent in file<span style="right: 14%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Exomiser_gene_pheno_score :</span> Absent in file<br><span class="commentTitle">Human_pheno_evidence :</span> Absent in file<br><span class="commentTitle">Mouse_pheno_evidence :</span> Absent in file<br><span class="commentTitle">Fish_pheno_evidence :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 28%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 42%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<br><span class="commentTitle">B_gain_coord :</span> .<br><span class="commentTitle">B_gain_source :</span> .<br><span class="commentTitle">B_gain_coord :</span> .</span></div>	</td>
	<td ><div class="tooltip">.<span style="right: 57%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<br><span class="commentTitle">P_gain_source :</span> .<br><span class="commentTitle">P_gain_phen :</span> .<br><span class="commentTitle">P_gain_hpo :</span> .<br><span class="commentTitle">P_gain_coord :</span> .<br><span class="commentTitle">P_gain_source :</span> .<br><span class="commentTitle">P_gain_phen :</span> .<br><span class="commentTitle">P_gain_hpo :</span> .</span></div>	</td>
	<td ><div class="tooltip">.<span style="right: 71%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 85%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="Absent in file"><div class="tooltip">Absent in file<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> Absent in file</span></div>	</td>
</tr>
<tr class="fullsplit 4_1563818_2158413_DUP_1" >
	<td data-order="31" data-search="4_1563818_2158413_DUP_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr4:1563818-2158413&hgt.out1=submit&highlight=hg19.chr4:1563818-2158413#aaedff" target="_blank" rel="noopener noreferrer"">4_1563818_2158413_DUP_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :<br></span> 4_1563818_2158413_DUP_1<br><span class="commentTitle">SV_length :</span> 594595</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 71%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ACMG_class :</span> .</span></div>	</td>
	<td data-search="DUP"><div class="tooltip">DUP<span style="left: 57%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DUP</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 42%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#3FFF00" data-search="TMEM129"><div class="tooltip"><a href="https://www.genenames.org/tools/search/#!/all?query=TMEM129" target="_blank" rel="noopener noreferrer"">TMEM129</a><span style="left: 28%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> TMEM129<br><span class="commentTitle">LOEUF_bin :</span> 8<br><span class="commentTitle">GnomAD_pLI :</span> 2.3074e-11<br><span class="commentTitle">ExAC_pLI :</span> 1.5746e-06<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> 60.14<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> 1.8774465029651<br><span class="commentTitle">ExAC_misZ :</span> 0.292414150336672<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 14%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_001127266<br><span class="commentTitle">Exon_count :</span> 4<br><span class="commentTitle">Overlapped_tx_length :</span> 5372<br><span class="commentTitle">Overlapped_CDS_length :</span> 1089<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_Nearest_SS :</span> .<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_Start :</span> 1717678<br><span class="commentTitle">Intersect_End :</span> 1723050</span></div>	</td>
	<td ><div class="tooltip">.<span style="left: 0%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidates :</span> .</span></div>	</td>
	<td data-search="Absent in file"><div class="tooltip">Absent in file<span style="right: 14%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Exomiser_gene_pheno_score :</span> Absent in file<br><span class="commentTitle">Human_pheno_evidence :</span> Absent in file<br><span class="commentTitle">Mouse_pheno_evidence :</span> Absent in file<br><span class="commentTitle">Fish_pheno_evidence :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 28%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 42%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<br><span class="commentTitle">B_gain_coord :</span> .<br><span class="commentTitle">B_gain_source :</span> .<br><span class="commentTitle">B_gain_coord :</span> .</span></div>	</td>
	<td ><div class="tooltip">.<span style="right: 57%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<br><span class="commentTitle">P_gain_source :</span> .<br><span class="commentTitle">P_gain_phen :</span> .<br><span class="commentTitle">P_gain_hpo :</span> .<br><span class="commentTitle">P_gain_coord :</span> .<br><span class="commentTitle">P_gain_source :</span> .<br><span class="commentTitle">P_gain_phen :</span> .<br><span class="commentTitle">P_gain_hpo :</span> .</span></div>	</td>
	<td ><div class="tooltip">.<span style="right: 71%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 85%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="Absent in file"><div class="tooltip">Absent in file<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> Absent in file</span></div>	</td>
</tr>
<tr class="fullsplit 4_1563818_2158413_DUP_1" >
	<td data-order="32" data-search="4_1563818_2158413_DUP_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr4:1563818-2158413&hgt.out1=submit&highlight=hg19.chr4:1563818-2158413#aaedff" target="_blank" rel="noopener noreferrer"">4_1563818_2158413_DUP_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :<br></span> 4_1563818_2158413_DUP_1<br><span class="commentTitle">SV_length :</span> 594595</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 71%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ACMG_class :</span> .</span></div>	</td>
	<td data-search="DUP"><div class="tooltip">DUP<span style="left: 57%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DUP</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 42%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFFFF" data-search="SCARNA22"><div class="tooltip"><a href="https://www.genenames.org/tools/search/#!/all?query=SCARNA22" target="_blank" rel="noopener noreferrer"">SCARNA22</a><span style="left: 28%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> SCARNA22<br><span class="commentTitle">LOEUF_bin :</span> .<br><span class="commentTitle">GnomAD_pLI :</span> .<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> .<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 14%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> UTR<br><span class="commentTitle">Tx :</span> NR_003004<br><span class="commentTitle">Exon_count :</span> 1<br><span class="commentTitle">Overlapped_tx_length :</span> 125<br><span class="commentTitle">Overlapped_CDS_length :</span> 0<br><span class="commentTitle">Overlapped_CDS_percent :</span> 0<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_Nearest_SS :</span> .<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_Start :</span> 1976362<br><span class="commentTitle">Intersect_End :</span> 1976487</span></div>	</td>
	<td ><div class="tooltip">.<span style="left: 0%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidates :</span> .</span></div>	</td>
	<td data-search="Absent in file"><div class="tooltip">Absent in file<span style="right: 14%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Exomiser_gene_pheno_score :</span> Absent in file<br><span class="commentTitle">Human_pheno_evidence :</span> Absent in file<br><span class="commentTitle">Mouse_pheno_evidence :</span> Absent in file<br><span class="commentTitle">Fish_pheno_evidence :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 28%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 42%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<br><span class="commentTitle">B_gain_coord :</span> .<br><span class="commentTitle">B_gain_source :</span> .<br><span class="commentTitle">B_gain_coord :</span> .</span></div>	</td>
	<td ><div class="tooltip">.<span style="right: 57%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<br><span class="commentTitle">P_gain_source :</span> .<br><span class="commentTitle">P_gain_phen :</span> .<br><span class="commentTitle">P_gain_hpo :</span> .<br><span class="commentTitle">P_gain_coord :</span> .<br><span class="commentTitle">P_gain_source :</span> .<br><span class="commentTitle">P_gain_phen :</span> .<br><span class="commentTitle">P_gain_hpo :</span> .</span></div>	</td>
	<td ><div class="tooltip">.<span style="right: 71%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 85%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="Absent in file"><div class="tooltip">Absent in file<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> Absent in file</span></div>	</td>
</tr>
<tr class="fullsplit 4_1563818_2158413_DUP_1" >
	<td data-order="33" data-search="4_1563818_2158413_DUP_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr4:1563818-2158413&hgt.out1=submit&highlight=hg19.chr4:1563818-2158413#aaedff" target="_blank" rel="noopener noreferrer"">4_1563818_2158413_DUP_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :<br></span> 4_1563818_2158413_DUP_1<br><span class="commentTitle">SV_length :</span> 594595</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 71%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ACMG_class :</span> .</span></div>	</td>
	<td data-search="DUP"><div class="tooltip">DUP<span style="left: 57%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DUP</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 42%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFFFF" data-search="MIR943"><div class="tooltip"><a href="https://www.genenames.org/tools/search/#!/all?query=MIR943" target="_blank" rel="noopener noreferrer"">MIR943</a><span style="left: 28%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> MIR943<br><span class="commentTitle">LOEUF_bin :</span> .<br><span class="commentTitle">GnomAD_pLI :</span> .<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> .<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> .<br><span class="commentTitle">DDD_mode :</span> .<br><span class="commentTitle">DDD_consequence :</span> .<br><span class="commentTitle">DDD_disease :</span> .<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 14%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> UTR<br><span class="commentTitle">Tx :</span> NR_030641<br><span class="commentTitle">Exon_count :</span> 1<br><span class="commentTitle">Overlapped_tx_length :</span> 94<br><span class="commentTitle">Overlapped_CDS_length :</span> 0<br><span class="commentTitle">Overlapped_CDS_percent :</span> 0<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_Nearest_SS :</span> .<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_Start :</span> 1988110<br><span class="commentTitle">Intersect_End :</span> 1988204</span></div>	</td>
	<td ><div class="tooltip">.<span style="left: 0%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidates :</span> .</span></div>	</td>
	<td data-search="Absent in file"><div class="tooltip">Absent in file<span style="right: 14%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Exomiser_gene_pheno_score :</span> Absent in file<br><span class="commentTitle">Human_pheno_evidence :</span> Absent in file<br><span class="commentTitle">Mouse_pheno_evidence :</span> Absent in file<br><span class="commentTitle">Fish_pheno_evidence :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 28%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 42%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<br><span class="commentTitle">B_gain_coord :</span> .<br><span class="commentTitle">B_gain_source :</span> .<br><span class="commentTitle">B_gain_coord :</span> .</span></div>	</td>
	<td ><div class="tooltip">.<span style="right: 57%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<br><span class="commentTitle">P_gain_source :</span> .<br><span class="commentTitle">P_gain_phen :</span> .<br><span class="commentTitle">P_gain_hpo :</span> .<br><span class="commentTitle">P_gain_coord :</span> .<br><span class="commentTitle">P_gain_source :</span> .<br><span class="commentTitle">P_gain_phen :</span> .<br><span class="commentTitle">P_gain_hpo :</span> .</span></div>	</td>
	<td ><div class="tooltip">.<span style="right: 71%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> .<br><span class="commentTitle">P_snvindel_phen :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 85%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="Absent in file"><div class="tooltip">Absent in file<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> Absent in file</span></div>	</td>
</tr>
<tr class="fullsplit 4_1563818_2158413_DUP_1" >
	<td data-order="34" data-search="4_1563818_2158413_DUP_1"><div class="tooltip"><a href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr4:1563818-2158413&hgt.out1=submit&highlight=hg19.chr4:1563818-2158413#aaedff" target="_blank" rel="noopener noreferrer"">4_1563818_2158413_DUP_1</a><span style="height: 469px; left: 1%" class="tooltiptext tooltip-bottom"><span class="commentTitle">AnnotSV_ID :<br></span> 4_1563818_2158413_DUP_1<br><span class="commentTitle">SV_length :</span> 594595</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="left: 71%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ACMG_class :</span> .</span></div>	</td>
	<td data-search="DUP"><div class="tooltip">DUP<span style="left: 57%" class="tooltiptext tooltip-bottom"><span class="commentTitle">SV_type :</span> DUP</span></div>	</td>
	<td data-search="split"><div class="tooltip">split<span style="left: 42%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Annotation_mode :</span> split</span></div>	</td>
	<td style="background-color:#FFFFFF" data-search="NSD2"><div class="tooltip"><a href="https://www.genenames.org/tools/search/#!/all?query=NSD2" target="_blank" rel="noopener noreferrer"">NSD2</a><span style="left: 28%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Gene_name :</span> NSD2<br><span class="commentTitle">LOEUF_bin :</span> .<br><span class="commentTitle">GnomAD_pLI :</span> .<br><span class="commentTitle">ExAC_pLI :</span> .<br><span class="commentTitle">HI :</span> .<br><span class="commentTitle">TS :</span> .<br><span class="commentTitle">DDD_HI_percent :</span> .<br><span class="commentTitle">ACMG :</span> .<br><span class="commentTitle">ExAC_cnvZ :</span> .<br><span class="commentTitle">ExAC_delZ :</span> .<br><span class="commentTitle">ExAC_dupZ :</span> .<br><span class="commentTitle">ExAC_synZ :</span> .<br><span class="commentTitle">ExAC_misZ :</span> .<br><span class="commentTitle">DDD_status :</span> probable<br><span class="commentTitle">DDD_mode :</span> monoallelic<br><span class="commentTitle">DDD_consequence :</span> loss of function<br><span class="commentTitle">DDD_disease :</span> NSD2-related developmental disorder (monoallelic)<br><span class="commentTitle">DDD_pmid :</span> .</span></div>	</td>
	<td data-search="txStart-txEnd"><div class="tooltip">txStart-txEnd<span style="left: 14%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Location :</span> txStart-txEnd<br><span class="commentTitle">Location2 :</span> 5'UTR-3'UTR<br><span class="commentTitle">Tx :</span> NM_001042424<br><span class="commentTitle">Exon_count :</span> 22<br><span class="commentTitle">Overlapped_tx_length :</span> 110800<br><span class="commentTitle">Overlapped_CDS_length :</span> 4098<br><span class="commentTitle">Overlapped_CDS_percent :</span> 100<br><span class="commentTitle">Frameshift :</span> no<br><span class="commentTitle">Dist_Nearest_SS :</span> .<br><span class="commentTitle">Nearest_SS_type :</span> .<br><span class="commentTitle">Intersect_Start :</span> 1873119<br><span class="commentTitle">Intersect_End :</span> 1983919</span></div>	</td>
	<td ><div class="tooltip">.<span style="left: 0%" class="tooltiptext tooltip-bottom"><span class="commentTitle">OMIM_phenotype :</span> .<br><span class="commentTitle">OMIM_ID :</span> .<br><span class="commentTitle">OMIM_inheritance :</span> .<br><span class="commentTitle">OMIM_morbid :</span> .<br><span class="commentTitle">OMIM_morbid_candidates :</span> .</span></div>	</td>
	<td data-search="Absent in file"><div class="tooltip">Absent in file<span style="right: 14%" class="tooltiptext tooltip-bottom"><span class="commentTitle">Exomiser_gene_pheno_score :</span> Absent in file<br><span class="commentTitle">Human_pheno_evidence :</span> Absent in file<br><span class="commentTitle">Mouse_pheno_evidence :</span> Absent in file<br><span class="commentTitle">Fish_pheno_evidence :</span> Absent in file</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 28%" class="tooltiptext tooltip-bottom"><span class="commentTitle">RE_gene :</span> .</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 42%" class="tooltiptext tooltip-bottom"><span class="commentTitle">B_gain_source :</span> .<br><span class="commentTitle">B_gain_coord :</span> .<br><span class="commentTitle">B_gain_source :</span> .<br><span class="commentTitle">B_gain_coord :</span> .</span></div>	</td>
	<td ><div class="tooltip">.<span style="right: 57%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_gain_coord :</span> .<br><span class="commentTitle">P_gain_source :</span> .<br><span class="commentTitle">P_gain_phen :</span> .<br><span class="commentTitle">P_gain_hpo :</span> .<br><span class="commentTitle">P_gain_coord :</span> .<br><span class="commentTitle">P_gain_source :</span> .<br><span class="commentTitle">P_gain_phen :</span> .<br><span class="commentTitle">P_gain_hpo :</span> .</span></div>	</td>
	<td ><div class="tooltip">9<span style="right: 71%" class="tooltiptext tooltip-bottom"><span class="commentTitle">P_snvindel_nb :</span> 9<br><span class="commentTitle">P_snvindel_phen :</span> 4p_partial_monosomy_syndrome; Inborn_genetic_diseases; Wolf-Hirschhorn_like_syndrome</span></div>	</td>
	<td data-search="."><div class="tooltip">.<span style="right: 85%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_left :</span> .</span></div>	</td>
	<td data-search="Absent in file"><div class="tooltip">Absent in file<span style="right: 100%" class="tooltiptext tooltip-bottom"><span class="commentTitle">ENCODE_blacklist_characteristics_right :</span> Absent in file</span></div>	</td>
</tr>
	</tbody>
	</table>
		</div>



	</body>
</html>